<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy Europe</title>
  <meta property="og:title" content="" />
  <meta property="og:description" content="" />
  <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" />
  <meta name="description" content="The European Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://galaxyproject.eu/index-virology.html">
  <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy Europe" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>

  <body>
    <div id="wrap">
      <div id="main">
        <div class="container" id="maincontainer">
          <div class="home">










<div class="alert alert-danger" style="margin-top: 0.5em">
	<h4>Peace to Ukraine!</h4>
	
	
	<div><p>üá∫üá¶ The global community has created a <a href="https://bit.ly/ua-table" target="_blank">continuously updated list</a> of laboratories that can host Ukrainian scientists at all career levels. If your lab can host a scientist ‚Äì add your name to the list <a href="https://bit.ly/ua-form" target="_blank">here</a>. In addition, Galaxy Project has a number of positions at its EU and US sites. Contact us at Contact us at <a href="mailto:ukraine@galaxyproject.org">ukraine@galaxyproject.org</a>. German organisations are organising accommodation <a href="https://elinor.network/gastfreundschaft-ukraine/" target="_blank">here</a>.</p>

<p>–ù–∞—É–∫–æ–≤–µ —Å—É—Å–ø—ñ–ª—å—Å—Ç–≤–æ —Å—Ç–≤–æ—Ä–∏–ª–æ –ø–æ—Å—Ç—ñ–π–Ω–æ –≤—ñ–¥–Ω–æ–≤–ª—é—î–º–∏–π <a href="https://bit.ly/ua-table" target="_blank">—Å–ø–∏—Å–æ–∫ –ª–∞–±–æ—Ä–∞—Ç–æ—Ä—ñ–π</a>, —è–∫—ñ –º–æ–∂—É—Ç—å –ø—Ä–∏–π–Ω—è—Ç–∏ —É–∫—Ä–∞—ó–Ω—Å—å–∫–∏—Ö –≤—á–µ–Ω–∏—Ö (–≤–∫–ª—é—á–∞—é—á–∏ –∞—Å–ø—ñ—Ä–∞–Ω—Ç—ñ–≤). –î–æ–¥–∞—Ç–∫–æ–≤–æ, Galaxy Project –º–∞—î –≤—ñ–¥–∫—Ä–∏—Ç—ñ –ø–æ–∑–∏—Ü—ñ—ó –Ω–∞ —Å–≤–æƒ´—Ö —î–≤—Ä–æ–ø–µ–π—Å—å–∫–∏—Ö —Ç–∞ –∞–º–µ—Ä–∏–∫–∞–Ω—Å—å–∫–∏—Ö —Å–∞–π—Ç–∞—Ö. –ö–æ–Ω—Ç–∞–∫—Ç—É–π—Ç–µ –Ω–∞—Å –≤–∏–∫–æ—Ä–∏—Å—Ç–æ–≤—É—è <a href="mailto:ukraine@galaxyproject.org">ukraine@galaxyproject.org</a></p>

<p>–ù–∞—É—á–Ω–æ–µ —Å–æ–æ–±—â–µ—Å—Ç–≤–æ —Å–æ–∑–¥–∞–ª–æ –ø–æ—Å—Ç–æ—è–Ω–Ω–æ –æ–±–Ω–æ–≤–ª—è–µ–º—ã–π <a href="https://bit.ly/ua-table" target="_blank">—Å–ø–∏—Å–æ–∫ –ª–∞–±–æ—Ä–∞—Ç–æ—Ä–∏–π</a>, –∫–æ—Ç–æ—Ä—ã–µ –º–æ–≥—É—Ç –ø—Ä–∏–Ω—è—Ç—å —É–∫—Ä–∞–∏–Ω—Å–∫–∏—Ö —É—á–µ–Ω—ã—Ö (–≤–∫–ª—é—á–∞—è –∞—Å–ø–∏—Ä–∞–Ω—Ç–æ–≤). –ö —Ç–æ–º—É –∂–µ, Galaxy Project –∏–º–µ–µ—Ç –æ—Ç–∫—Ä—ã—Ç—ã–µ –ø–æ–∑–∏—Ü–∏–∏ –Ω–∞ —Å–≤–æ–∏—Ö –µ–≤—Ä–æ–ø–µ–π—Å–∫–∏—Ö –∏ –∞–º–µ—Ä–∏–∫–∞–Ω—Å–∫–∏—Ö —Å–∞–π—Ç–∞—Ö.  –ö–æ–Ω—Ç–∞–∫—Ç–∏—Ä—É–π—Ç–µ –Ω–∞—Å –∏—Å–ø–æ–ª—å–∑—É—è <a href="mailto:ukraine@galaxyproject.org">ukraine@galaxyproject.org</a> üá∫üá¶</p>
</div>
	
</div>








<div class="alert alert-info" style="margin-top: 0.5em">
	<h4>Limited computing capacity on next 01.03.2022</h4>
	
	
	<div><p>UPDATE 04.03.2022 09:00 CET, bwCloud maintenace is still on going.</p>

<p>UPDATE 03.03.2022 11:00 CET, Usegalaxy.eu is still without compute capacity. We hope to return to normal operation soon.</p>

<p>On March 1st, bwCloud will undergo a scheduled maintenance, and we have to shut down most of our computing resources.
We will keep the web server running, accepting jobs and workflows, but there will be a delay in processing them.
The running jobs, at that date, will be interrupted and rescheduled for a new start (from scratch, no intermediate results can be saved) 
as soon as the cloud is available again.
Please take it into account in your job schedule.</p>

<p>Sorry for the inconvenience.</p>
</div>
	
</div>





  <h1 class="no_toc" id="welcome-to-the-european-virology-galaxy-flavor">Welcome to the European Virology Galaxy flavor</h1>

<p><img src="/assets/media/virology_workbench.jpeg" alt="Virology Galaxy" class="rna-intro-right" /></p>

<p>The Virology Galaxy workbench is a comprehensive set of analysis tools and consolidated workflows.
The workbench is based on the Galaxy framework, which guarantees simple access, easy extension, flexible adaption to personal and security needs,
and sophisticated analyses independent of command-line knowledge.</p>

<h1 class="no_toc" id="content">Content</h1>

<ol id="markdown-toc">
  <li><a href="#the-sars-cov-2-project" id="markdown-toc-the-sars-cov-2-project">The SARS-CoV-2 project</a>    <ol>
      <li><a href="#preprocessing-of-raw-sars-cov-2-reads" id="markdown-toc-preprocessing-of-raw-sars-cov-2-reads">Preprocessing of raw SARS-CoV-2 reads</a></li>
      <li><a href="#assembly-of-sars-cov-2-from-pre-processed-reads" id="markdown-toc-assembly-of-sars-cov-2-from-pre-processed-reads">Assembly of SARS-CoV-2 from pre-processed reads</a></li>
      <li><a href="#dating-the-most-common-recent-ancestor-mcra-of-sars-cov-2" id="markdown-toc-dating-the-most-common-recent-ancestor-mcra-of-sars-cov-2">Dating the most common recent ancestor (MCRA) of SARS-CoV-2</a></li>
      <li><a href="#analysis-of-variation-within-individual-covid-19-samples" id="markdown-toc-analysis-of-variation-within-individual-covid-19-samples">Analysis of variation within individual COVID-19 samples</a></li>
      <li><a href="#history-and-workflow-2" id="markdown-toc-history-and-workflow-2">History and workflow</a></li>
      <li><a href="#alignment-of-covid-19-spike-protein-with-homologs-from-other-coronaviruses" id="markdown-toc-alignment-of-covid-19-spike-protein-with-homologs-from-other-coronaviruses">Alignment of COVID-19 Spike protein with homologs from other coronaviruses</a></li>
      <li><a href="#evolutionary-analysis" id="markdown-toc-evolutionary-analysis">Evolutionary Analysis</a></li>
      <li><a href="#inputs-4" id="markdown-toc-inputs-4">Inputs</a></li>
      <li><a href="#comparative-analysis-of-coronovirus-sequences" id="markdown-toc-comparative-analysis-of-coronovirus-sequences">Comparative analysis of coronovirus sequences</a></li>
    </ol>
  </li>
  <li><a href="#training" id="markdown-toc-training">Training</a></li>
  <li><a href="#available-tools" id="markdown-toc-available-tools">Available tools</a>    <ol>
      <li><a href="#rna-structure-prediction-and-analysis" id="markdown-toc-rna-structure-prediction-and-analysis">RNA structure prediction and analysis</a></li>
      <li><a href="#rna-alignment" id="markdown-toc-rna-alignment">RNA alignment</a></li>
      <li><a href="#rna-annotation" id="markdown-toc-rna-annotation">RNA annotation</a></li>
      <li><a href="#rna-protein-interaction" id="markdown-toc-rna-protein-interaction">RNA-protein interaction</a></li>
      <li><a href="#rna-rna-interaction" id="markdown-toc-rna-rna-interaction">RNA-RNA interaction</a></li>
      <li><a href="#rna-target-prediction" id="markdown-toc-rna-target-prediction">RNA target prediction</a></li>
      <li><a href="#rna-seq-and-hts-analysis" id="markdown-toc-rna-seq-and-hts-analysis">RNA Seq and HTS analysis</a></li>
      <li><a href="#ribosome-profiling" id="markdown-toc-ribosome-profiling">Ribosome profiling</a></li>
    </ol>
  </li>
  <li><a href="#contributors" id="markdown-toc-contributors">Contributors</a></li>
</ol>

<h1 id="the-sars-cov-2-project">The <a href="https://github.com/galaxyproject/SARS-CoV-2">SARS-CoV-2</a> project</h1>

<p>Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take
<a href="https://virology.usegalaxy.eu/tours/core.galaxy_ui" target="_blank">a guided tour</a> through Galaxy‚Äôs user interface.</p>

<p>All data that you need is available on <a href="https://github.com/galaxyproject/SARS-CoV-2">GitHub</a> or in our dedicated <a href="https://usegalaxy.eu/library/list#folders/Fbe28a3bea3980e71">data library</a>.
Otherwise, we will start with the preprocessing of raw <a href="https://github.com/galaxyproject/SARS-CoV-2">SARS-CoV-2</a> reads.</p>

<h2 id="preprocessing-of-raw-sars-cov-2-reads">Preprocessing of raw SARS-CoV-2 reads</h2>

<p>The raw reads available so far are generated from bronchoalveolar lavage fluid (BALF) and are metagenomic in nature: they contain human reads, reads from potential bacterial co-infections as well as true COVID-19 reads.</p>

<h3 id="whats-the-point">What‚Äôs the point?</h3>

<p>Assess quality of reads, remove adapters and remove reads mapping to human genome.</p>

<h3 id="the-outline">The outline</h3>

<p>Illumina and Oxford nanopore reads are pulled from the NCBI SRA (links to SRA accessions are available <a href="https://www.ncbi.nlm.nih.gov/genbank/sars-cov-2-seqs/">here</a>). They are then processed separately as described in the <a href="#the-history-and-the-workflow">workflow section</a>.</p>

<h3 id="inputs">Inputs</h3>

<p>Only SRA accessions are required for this analysis. The described analysis was performed with all SRA SARS-CoV accessions available as of Feb 20, 2020:</p>

<ol>
  <li>
    <p>Illumina reads</p>

    <div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>SRR10903401
SRR10903402
SRR10971381
</code></pre></div>    </div>
  </li>
  <li>
    <p>Oxford Nanopore reads</p>

    <div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>SRR10948550
SRR10948474
SRR10902284
</code></pre></div>    </div>
  </li>
</ol>

<h3 id="outputs">Outputs</h3>

<p>This workflow produces three outputs that are used in tow subsequent analyses</p>

<table class="table table-striped">
  <thead>
    <tr>
      <th><code class="highlighter-rouge">#</code></th>
      <th>Output</th>
      <th>Used in</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>A combined set of adapter-free Illumina reads without human contamination</td>
      <td><a href="https://github.com/galaxyproject/SARS-CoV-2/tree/master/2-Assembly">Assembly</a></td>
    </tr>
    <tr>
      <td>2</td>
      <td>A combined set of Oxford Nanopore reads without human contamination</td>
      <td><a href="https://github.com/galaxyproject/SARS-CoV-2/tree/master/2-Assembly">Assembly</a></td>
    </tr>
    <tr>
      <td>3</td>
      <td>A collection of adapter-free Illumina reads from which human reads <em>have not</em> been removed</td>
      <td><a href="https://github.com/galaxyproject/SARS-CoV-2/tree/master/4-Variation">Variation detection</a></td>
    </tr>
  </tbody>
</table>

<h3 id="the-history-and-the-workflow">The history and the workflow</h3>

<p>A Galaxy workspace (history) containing the most current analysis can be imported from <a href="https://usegalaxy.org/u/aun1/h/covid-19-pre-processing">here</a>.</p>

<p>The publicly accessible <a href="https://usegalaxy.org/u/aun1/w/covid-19-pre-pp">workflow</a> can be downloaded and installed on any Galaxy instance. It contains version information for all tools used in this analysis.</p>

<p>The workflow performs the following steps:</p>

<h4 id="illumina">Illumina</h4>

<ul>
  <li>Illumina reads are QC‚Äôed and adapter sequences are removed using <code class="highlighter-rouge">fastp</code></li>
  <li>Quality metrics are computed and visualized using <code class="highlighter-rouge">fastqc</code>  and <code class="highlighter-rouge">multiqc</code></li>
  <li>Reads are mapped against human genome version <code class="highlighter-rouge">hg38</code> using <code class="highlighter-rouge">bwa mem</code></li>
  <li>Reads that <strong>do not map</strong> to <code class="highlighter-rouge">hg38</code> are filtered out using <code class="highlighter-rouge">samtools view</code></li>
  <li>Reads are converted back to fastq format using <code class="highlighter-rouge">samtools fastx</code></li>
</ul>

<h4 id="oxford-nanopore">Oxford nanopore</h4>

<ul>
  <li>Reads are QC‚Äôed using <code class="highlighter-rouge">nanoplot</code></li>
  <li>Quality metrics are computed and visualized using <code class="highlighter-rouge">fastqc</code>  and <code class="highlighter-rouge">multiqc</code></li>
  <li>Reads are mapped against human genome version <code class="highlighter-rouge">hg38</code> using <code class="highlighter-rouge">minimap2</code></li>
  <li>Reads that <strong>do not map</strong> to <code class="highlighter-rouge">hg38</code> are filtered out using <code class="highlighter-rouge">samtools view</code></li>
  <li>Reads are converted back to fastq format using <code class="highlighter-rouge">samtools fastx</code></li>
</ul>

<p><img src="/assets/media/sars/pp_wf.png" alt="" height="50%" width="50%" /></p>

<h2 id="assembly-of-sars-cov-2-from-pre-processed-reads">Assembly of SARS-CoV-2 from pre-processed reads</h2>

<h3 id="whats-the-point-1">What‚Äôs the point?</h3>

<p>Use a combination of Illumina and Oxford Nanopore reads to produce SARS-CoV-2 genome assembly.</p>

<h3 id="outline">Outline</h3>

<p>We use Illumina and Oxford Nanopore reads that were pre-processed to remove human-derived sequences. We use two assembly tools: <a href="http://cab.spbu.ru/software/spades/"><code class="highlighter-rouge">spades</code></a> and <a href="https://github.com/rrwick/Unicycler"><code class="highlighter-rouge">unicycler</code></a>. While <code class="highlighter-rouge">spades</code> is a tool fully dedicated to assembly, <code class="highlighter-rouge">unicycler</code> is a ‚Äúwrapper‚Äù that combines multiple existing tools. It uses <code class="highlighter-rouge">spades</code> as an engine for short read assembly while utilizing <a href="https://github.com/lh3/miniasm"><code class="highlighter-rouge">mimiasm</code></a> and <a href="https://github.com/isovic/racon"><code class="highlighter-rouge">racon</code></a> for assembly of long noisy reads.</p>

<p>In addition to assemblies (actual sequences) the two tools produce assembly graphs that can be used for visualization of assembly with <a href="https://rrwick.github.io/Bandage/"><code class="highlighter-rouge">bandage</code></a>.</p>

<h3 id="inputs-1">Inputs</h3>

<p>Filtered Illumina and Oxford Nanopore reads produced during the <a href="https://github.com/galaxyproject/SARS-CoV-2/tree/master/PreProcessing">pre-processing step</a> are used as inputs to the assembly tools.</p>

<h3 id="outputs-1">Outputs</h3>

<p>Each tool produces assembly (contigs) and assembly graph representations. The largest contigs generated by <code class="highlighter-rouge">unicycler</code> and <code class="highlighter-rouge">spades</code> were 29,781 and 29,907 nts, respectively, and had 100% identity over their entire length.</p>

<p>The following figures show visualizations of assembly graphs produced with <code class="highlighter-rouge">spades</code> and <code class="highlighter-rouge">unicycler</code>. The complexity of the graphs is not surprising given the metagenomic nature of the underlying samples.</p>

<table>
  <thead>
    <tr>
      <th style="text-align: left">Assembly graphs for Unicycler (A) and SPAdes (B)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: left"><img src="https://usegalaxy.org/datasets/bbd44e69cb8906b5d6265148ad20e586/display/?preview=True" alt="" height="50%" width="50%" /></td>
    </tr>
    <tr>
      <td style="text-align: left"><strong>A</strong>. Unicycler assembly graph</td>
    </tr>
    <tr>
      <td style="text-align: left"><img src="https://usegalaxy.org/datasets/bbd44e69cb8906b5f5dc8f4de00733be/display/?preview=True" alt="" height="50%" width="50%" /></td>
    </tr>
    <tr>
      <td style="text-align: left"><strong>B</strong>. SPAdes assembly graph</td>
    </tr>
  </tbody>
</table>

<h3 id="history-and-workflow">History and workflow</h3>

<p>A Galaxy workspace (history) containing the most current analysis can be imported from <a href="https://usegalaxy.org/u/aun1/h/ncov-assembly">here</a>.</p>

<p>The publicly accessible <a href="https://usegalaxy.org/u/aun1/w/ncov-assembly-1">workflow</a> can be downloaded and installed on any Galaxy instance. It contains version information for all tools used in this analysis.</p>

<p><img src="/assets/media/sars/as_wf.png" alt="" height="50%" width="50%" /></p>

<h2 id="dating-the-most-common-recent-ancestor-mcra-of-sars-cov-2">Dating the most common recent ancestor (MCRA) of SARS-CoV-2</h2>

<h3 id="whats-the-point-2">What‚Äôs the point?</h3>

<p>For this we used simple root-to-tip regression <a href="https://www.ncbi.nlm.nih.gov/pubmed/10846155">Korber et al. 2000</a> (more complex and powerful phylodynamics methods could certainly be used, but for this data with very low levels of sequence divergence, simpler and faster methods suffice). Using a set of sequences from all COVID-19 sequences available as of Feb 16, 2020 we obtained an MCRA date of Nov 14, 2019, which is close to other existing estimates <a href="http://virological.org/t/phylodynamic-analysis-115-genomes-20-feb-2020/356">Rambaut 2020</a>.</p>

<h3 id="outline-1">Outline</h3>

<p>This analysis consists of two components - a Galaxy workflow and a Jupyter notebook.</p>

<p>The workflow is used to extract full length sequences of SARS-CoV-2, tidy up their names in FASTA files, produce a multiple sequences alignment and compute a maximum likelihood tree.</p>

<p>The <a href="./MCRA_Estimation_Notebook.ipynb">Jupyter notebook</a> is used to correlate branch lengths with collection dates in order to estimate MCRA timing.</p>

<h3 id="inputs-2">Inputs</h3>

<p>One input is required: a comma-separated <a href="acc_date.txt">file</a> containing accession numbers and collection dates:</p>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>Accession,Collection_Date
MT019531,2019-12-30
MT019529,2019-12-23
MT007544,2020-01-25
MN975262,2020-01-11
...
</code></pre></div></div>

<p>An up-to-date version of this file can be generated directly from the <a href="https://www.ncbi.nlm.nih.gov/labs/virus/">NCBI Virus</a> resource by</p>

<ol>
  <li>searching for SARS-CoV-2 (NCBI taxid: 2697049) sequences</li>
  <li>configuring the list of results to display only the <code class="highlighter-rouge">Accession</code> and <code class="highlighter-rouge">Collection date</code> columns</li>
  <li>downloading the <code class="highlighter-rouge">Current table view result</code> in <code class="highlighter-rouge">CSV format</code></li>
</ol>

<p>The collection dates will be taken from the corresponding GenBank record‚Äôs <code class="highlighter-rouge">/collection_date</code> tag.</p>

<h3 id="outputs-2">Outputs</h3>

<h3 id="history-and-workflow-1">History and workflow</h3>

<p>A Galaxy workspace (history) containing the most current analysis can be imported from <a href="https://usegalaxy.org/u/aun1/h/ncov-mcra-timing">here</a>.</p>

<p>The publicly accessible <a href="https://usegalaxy.org/u/aun1/w/mcra">workflow</a> can be downloaded and installed on any Galaxy instance. It contains version information for all tools used in this analysis.</p>

<p><img src="/assets/media/sars/mcra_wf.png" alt="" height="50%" width="50%" /></p>

<h2 id="analysis-of-variation-within-individual-covid-19-samples">Analysis of variation within individual COVID-19 samples</h2>

<h3 id="whats-the-point-3">What‚Äôs the point?</h3>

<p>To understand the amount of heterogeneity in individual COVID-19 isolates.</p>

<h3 id="outline-2">Outline</h3>

<p>As of writing (2/13/2020) there were just three Illumina datasets from COVID-19 patients:</p>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>- sra-study: SRP242226
  bioproject: PRJNA601736
  biosample: SAMN13872787
  sra-sample: SRS6007144
  sra-experiment: SRX7571571
  sra-run: SRR10903401

- sra-study: SRP242226
  bioproject: PRJNA601736
  biosample: SAMN13872786
  sra-sample: SRS6007143
  sra-experiment: SRX7571570
  sra-run: SRR10903402

- sra-study: SRP245409
  bioproject: PRJNA603194
  biosample: SAMN13922059
  sra-sample: SRS6067521
  sra-experiment: SRX7636886
  sra-run: SRR10971381
</code></pre></div></div>

<p>To understand the extent of sequence variation within these samples we performed the following analysis. First, we used a Galaxy workflow to perform the following steps:</p>

<ol>
  <li>Mapped all reads against COVID-19 reference <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_045512">NC_045512.2</a> using <code class="highlighter-rouge">bwa mem</code></li>
  <li>Filtered reads with mapping quality of at least 20, that were mapped as proper pairs</li>
  <li>Performed realignments using <code class="highlighter-rouge">lofreq viterbi</code></li>
  <li>Called variants using <code class="highlighter-rouge">lofreq call</code></li>
  <li>Annotated variants using <code class="highlighter-rouge">snpeff</code> against database created from NC_045512.2 GenBank file</li>
  <li>Converted VCFs into tab delimited datasets</li>
</ol>

<p>Next, we analyzed this tab delimited data in a <a href="variation_analysis.ipynb">Jupyter notebook</a>.</p>

<h3 id="inputs-3">Inputs</h3>

<h4 id="workflow">Workflow</h4>

<ol>
  <li>
    <p>GenBank file for the reference COVID-19 <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_045512">genome</a>.</p>

    <p>The GenBank record is used by <code class="highlighter-rouge">snpeff</code> to generate a database for variant annotation.</p>
  </li>
  <li>
    <p>Set of illumina reads (in this case a collection of unfiltered reads from <code class="highlighter-rouge">SRR10903401</code>, <code class="highlighter-rouge">SRR10903402</code>, and <code class="highlighter-rouge">SRR10971381</code>)</p>
  </li>
</ol>

<h4 id="jupyter-notebook">Jupyter notebook</h4>

<p>The Jupyter notebook requires the GenBank file (#1 from above) and the output of the workflow described below.</p>

<h3 id="outputs-3">Outputs</h3>

<p>The workflow produces a table of variants that looks like this:</p>

<div>

<table class="table table-striped">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>Sample</th>
      <th>CHROM</th>
      <th>POS</th>
      <th>REF</th>
      <th>ALT</th>
      <th>DP</th>
      <th>AF</th>
      <th>SB</th>
      <th>DP4</th>
      <th>IMPACT</th>
      <th>FUNCLASS</th>
      <th>EFFECT</th>
      <th>GENE</th>
      <th>CODON</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>0</td>
      <td>SRR10903401</td>
      <td>NC_045512</td>
      <td>1409</td>
      <td>C</td>
      <td>T</td>
      <td>124</td>
      <td>0.040323</td>
      <td>1</td>
      <td>66,53,2,3</td>
      <td>MODERATE</td>
      <td>MISSENSE</td>
      <td>NON_SYNONYMOUS_CODING</td>
      <td>orf1ab</td>
      <td>Cat/Tat</td>
    </tr>
    <tr>
      <td>1</td>
      <td>SRR10903401</td>
      <td>NC_045512</td>
      <td>1821</td>
      <td>G</td>
      <td>A</td>
      <td>95</td>
      <td>0.094737</td>
      <td>0</td>
      <td>49,37,5,4</td>
      <td>MODERATE</td>
      <td>MISSENSE</td>
      <td>NON_SYNONYMOUS_CODING</td>
      <td>orf1ab</td>
      <td>gGt/gAt</td>
    </tr>
    <tr>
      <td>2</td>
      <td>SRR10903401</td>
      <td>NC_045512</td>
      <td>1895</td>
      <td>G</td>
      <td>A</td>
      <td>107</td>
      <td>0.037383</td>
      <td>0</td>
      <td>51,52,2,2</td>
      <td>MODERATE</td>
      <td>MISSENSE</td>
      <td>NON_SYNONYMOUS_CODING</td>
      <td>orf1ab</td>
      <td>Gta/Ata</td>
    </tr>
    <tr>
      <td>3</td>
      <td>SRR10903401</td>
      <td>NC_045512</td>
      <td>2407</td>
      <td>G</td>
      <td>T</td>
      <td>122</td>
      <td>0.024590</td>
      <td>0</td>
      <td>57,62,1,2</td>
      <td>MODERATE</td>
      <td>MISSENSE</td>
      <td>NON_SYNONYMOUS_CODING</td>
      <td>orf1ab</td>
      <td>aaG/aaT</td>
    </tr>
    <tr>
      <td>4</td>
      <td>SRR10903401</td>
      <td>NC_045512</td>
      <td>3379</td>
      <td>A</td>
      <td>G</td>
      <td>121</td>
      <td>0.024793</td>
      <td>0</td>
      <td>56,62,1,2</td>
      <td>LOW</td>
      <td>SILENT</td>
      <td>SYNONYMOUS_CODING</td>
      <td>orf1ab</td>
      <td>gtA/gtG</td>
    </tr>
  </tbody>
</table>
</div>

<p>Here, most fields names are descriptive. <strong>SB</strong> = the Phred-scaled probability of strand bias as calculated by
<a href="https://csb5.github.io/lofreq/">lofreq</a> (0 = no strand bias); <strong>DP4</strong> = strand-specific depth for reference and alternate allele observations
(Forward reference, reverse reference, forward alternate, reverse alternate).</p>

<hr />

<p>The variants we identified were distributed across the SARS-CoV-2 genome in the following way:</p>

<p><img src="/assets/media/sars/var_map.png" alt="" /></p>

<p>The following table describes variants with frequencies above 10%:</p>

<p><img src="/assets/media/sars/S_var.png" alt="" /></p>

<h2 id="history-and-workflow-2">History and workflow</h2>

<p>A Galaxy workspace (history) containing the most current analysis can be imported from <a href="https://usegalaxy.org/u/aun1/h/ncov-intrasample-variation">here</a>.</p>

<p>The publicly accessible <a href="https://usegalaxy.org/u/aun1/w/ncov-variation-analysis">workflow</a> can be downloaded and installed on any Galaxy instance.
It contains version information for all tools used in this analysis.</p>

<p><img src="/assets/media/sars/var_wf.png" alt="" height="50%" width="50%" /></p>

<h2 id="alignment-of-covid-19-spike-protein-with-homologs-from-other-coronaviruses">Alignment of COVID-19 Spike protein with homologs from other coronaviruses</h2>

<h3 id="whats-the-point-4">What‚Äôs the point?</h3>

<p>Aligning Spike protein sequences to detect structural variations and impact of polymorphisms.</p>

<h3 id="outline-3">Outline</h3>

<p>We generate a codon alignment for a set of coronaviruses in order to track polymorphisms uncovered by the analysis of <a href="https://github.com/galaxyproject/SARS-CoV-2/tree/master/Variation">variation in individual samples</a>.</p>

<h3 id="input">Input</h3>

<p>Downloaded CDS sequences of coronavirus Spike proteins from <a href="https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/virus?SeqType_s=Nucleotide&amp;VirusLineage_ss=SARS-CoV-2,%20taxid:2697049">NCBI Viral Resource</a> for the following coronaviruses:</p>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>FJ588692.1	Bat SARS Coronavirus Rs806/2006
KR559017.1	Bat SARS-like coronavirus BatCoV/BB9904/BGR/2008
KC881007.1	Bat SARS-like coronavirus WIV1
KT357810.1	MERS coronavirus isolate Riyadh_1175/KSA/2014
KT357811.1	MERS coronavirus isolate Riyadh_1337/KSA/2014
KT357812.1	MERS coronavirus isolate Riyadh_1340/KSA/2014
KF811036.1	MERS coronavirus strain Tunisia-Qatar_2013
AB593383.1	Murine hepatitis virus
AF190406.1	Murine hepatitis virus strain TY
AY687355.1	SARS coronavirus A013
AY687356.1	SARS coronavirus A021
AY687361.1	SARS coronavirus B029
AY687365.1	SARS coronavirus C013
AY687368.1	SARS coronavirus C018
AY648300.1	SARS coronavirus HHS-2004
DQ412594.1	SARS coronavirus isolate CUHKtc10NP
DQ412596.1	SARS coronavirus isolate CUHKtc14NP
DQ412609.1	SARS coronavirus isolate CUHKtc32NP
MN996528.1	nCov-2019
MN996527.1	nCov-2019
NC_045512.2	nCov-2019
NC_002306.3	Feline infectious peritonitis virus
NC_028806.1	Swine enteric coronavirus strain Italy/213306/2009
NC_038861.1	Transmissible gastroenteritis virus
</code></pre></div></div>

<h3 id="output">Output</h3>

<p>We produce two alignments, one at the nucleotide and one at the amino acid level, of Betacoronavirus spike proteins. The alignments can be visualized with the <code class="highlighter-rouge">Multiple Sequence Alignment</code> visualization in Galaxy :</p>

<table>
  <thead>
    <tr>
      <th style="text-align: center">¬†</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center"><img src="/assets/media/sars/align_galaxy_viz.png" alt="Visualization of amino acid alignment in Galaxy" height="50%" width="50%" /></td>
    </tr>
  </tbody>
</table>

<p>Or with locally installed softwares, here <a href="https://github.com/AliView/AliView"><code class="highlighter-rouge">AliView</code></a>.</p>

<table>
  <thead>
    <tr>
      <th style="text-align: center">Alignments of Spike proteins</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center"><img src="/assets/media/sars/Spike_CDS_Alignment.png" alt="Nucleic Alignment of Spike proteins" height="50%" width="50%" /></td>
    </tr>
    <tr>
      <td style="text-align: center"><strong>A</strong>. CDS alignments</td>
    </tr>
    <tr>
      <td style="text-align: center"><img src="/assets/media/sars/Spike_Protein_Alignment.png" alt="Proteic Alignment of Spike proteins" height="50%" width="50%" /></td>
    </tr>
    <tr>
      <td style="text-align: center"><strong>B</strong>. Protein alignment</td>
    </tr>
  </tbody>
</table>

<h3 id="workflow-1">Workflow</h3>

<p>The Galaxy history containing the latest analysis can be found <a href="https://usegalaxy.org/u/delphinel/h/s-proteins-alignment">here</a>. The publicly accessible <a href="https://usegalaxy.org/u/delphinel/w/cov-alignment">workflow</a> can be downloaded and installed on any Galaxy instance. It contains all information about tool versions and parameters used in this analysis.</p>

<p><img src="/assets/media/sars/Workflow_snapshot.png" alt="Analysis Workflow" height="50%" width="50%" /></p>

<p>The <code class="highlighter-rouge">Transeq</code> tool converts the CDS sequences into protein sequences, which we then align to each other using <code class="highlighter-rouge">MAFFT</code>. The output is fed into <code class="highlighter-rouge">tranalign</code> along with the nucleotide sequences. <code class="highlighter-rouge">tranalign</code> produces a nucleotide alignment coherent with the protein alignment.</p>

<h2 id="evolutionary-analysis">Evolutionary Analysis</h2>

<h3 id="whats-the-point-5">What‚Äôs the point?</h3>

<p><a href="https://doi.org/10.1038/s41586-020-2008-3">Wu et al.</a> showed recombination between COVID-19 and bat coronaviruses located within the <em>S</em>-gene. We want to confirm this observation and provide a publicly accessible workflow for recombination detection.</p>

<p>In previous coronovirus outbreaks (SARS), retrospecive analyses determined that adaptive substitutions might have occured in the S-protein <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1609170/">Zhang et al.</a>, e.g., related to <a href="https://www.embopress.org/doi/full/10.1038/sj.emboj.7600640">ACE2 receptor utilization</a>. While data on COVID-19 are currenly limited, we investigated whether or not the lineage leading to them showed any evidence of positive diversifying selection.</p>

<h3 id="outline-4">Outline</h3>

<p>We employ a recombination detection algorithm (GARD) developed by <a href="http://mbe.oxfordjournals.org/cgi/content/full/23/10/1891">Kosakovsky Pond et al.</a> and implemented in the <code class="highlighter-rouge">hyphy</code> package. To select a representative set of <em>S</em>-genes we perform a blast search using the <em>S</em>-gene CDS from <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_045512">NC_045512</a> as a query against the <code class="highlighter-rouge">nr</code> database. We select coding regions corresponding to the <em>S</em>-gene from a number of COVID-19 genomes, original SARS isolates. This set of sequences can be found in <a href="S_nt.fna">this repository</a></p>

<p>We then generate a codon-based alignment using the workflow shown below and perform the recombination analysis using the <code class="highlighter-rouge">gard</code> tool from the <code class="highlighter-rouge">hyphy</code> package.</p>

<p>For selection analyses, we apply the <a href="https://www.ncbi.nlm.nih.gov/pubmed/25697341">Adaptive Branch Site Random Effects</a> method to test whether or each branch of the tree shows evidence of diversifing positive selection along a fraction of sites using the <code class="highlighter-rouge">absrel</code> tool from the <code class="highlighter-rouge">hyphy</code> package.</p>

<h2 id="inputs-4">Inputs</h2>

<p>A set of unaligned CDS sequences for the <em>S</em>-gene.</p>

<p>Additionally, for aBSREL, a phylogenetic tree (for aBSREL).</p>

<h3 id="outputs-4">Outputs</h3>

<p>A recombination report:</p>

<p><img src="/assets/media/sars/dm_report.png" alt="" height="50%" width="50%" /></p>

<p>and a map of possible recombination hotspots:</p>

<p><img src="/assets/media/sars/dm_chart.png" alt="" height="50%" width="50%" /></p>

<p>A selection analysis summary and tree (COVID-19 isolate is MN988668_1)</p>

<p><img src="/assets/media/sars/dm_tree.png" alt="" height="50%" width="50%" /></p>

<p>and a plot of the inferred œâ distribution for the MN988668_1 branch.</p>

<p><img src="/assets/media/sars/dm_selection.png" alt="" height="50%" width="50%" /></p>

<h3 id="history-and-workflow-3">History and workflow</h3>

<blockquote>
  <p>TODO: add aBSREL workflow</p>
</blockquote>

<p>A Galaxy workspace (history) containing the most current analysis can be imported from <a href="https://usegalaxy.org/u/aun1/h/ncov-comp">here</a>.</p>

<p>The publicly accessible <a href="https://test.galaxyproject.org/u/anton/h/ncov-recomb">workflow</a> can be downloaded and installed on any Galaxy instance. It contains version information for all tools used in this analysis.</p>

<p><img src="/assets/media/sars/rec_wf.png" alt="" height="50%" width="50%" /></p>

<p>The workflow takes unaligned CDS sequences, translates them with <code class="highlighter-rouge">EMBOSS:tanseq</code>, aligns translations using <code class="highlighter-rouge">mafft</code>, realigns original CDS input using the mafft alignment as a guide and sends this codon-based alignment to <code class="highlighter-rouge">gard</code>.</p>

<h2 id="comparative-analysis-of-coronovirus-sequences">Comparative analysis of coronovirus sequences</h2>

<h3 id="whats-the-point-6">What‚Äôs the point?</h3>

<p>What is the phylogenetic relationship between assembled sequences and other coronaviruses?</p>

<h3 id="outline-5">Outline</h3>

<p>We mapped Unicycler assembly produces at <a href="https://github.com/galaxyproject/SARS-CoV-2/tree/master/Assembly">step 1</a> against <code class="highlighter-rouge">nr</code> database at NCBI using <code class="highlighter-rouge">blastn</code>  and downloaded <a href="4GRC05K5014-Alignment-HitTable.csv">hit table</a>. This analysis indicated that our assembly is 100% identical to <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_045512">NC_045512</a>.</p>

<p>We then used this hit table to Galaxy workflow that:</p>

<ol>
  <li>downloaded sequences</li>
  <li>aligned downloaded sequences against COVID-19 reference <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_045512">NC_045512</a> using <code class="highlighter-rouge">lastz</code></li>
  <li>identified sequences that align with at least 75% of NC_045512</li>
  <li>created multiple alignment of sequences from the previous step using <code class="highlighter-rouge">mafft</code></li>
  <li>computed a maximum likelihood tree using <code class="highlighter-rouge">iqtree</code></li>
</ol>

<h3 id="inputs-5">Inputs</h3>

<p>The analysis takes two inputs:</p>

<ol>
  <li>Hit table generated by blast</li>
  <li>Genbank file for COVID-19 reference genome <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_045512">NC_045512</a></li>
</ol>

<p>The hit table has the following format:</p>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>COVID-19,MN988668.1,100.000,29781,0,0,1,29781,29838,58,0.0,54996
COVID-19,NC_045512.2,100.000,29781,0,0,1,29781,29839,59,0.0,54996
COVID-19,MN994468.1,99.993,29781,2,0,1,29781,29839,59,0.0,54985
COVID-19,MN985325.1,99.990,29781,3,0,1,29781,29839,59,0.0,54979
COVID-19,MN938384.1,99.990,29781,3,0,1,29781,29807,27,0.0,54979
COVID-19,MN997409.1,99.987,29781,4,0,1,29781,29839,59,0.0,54974
COVID-19,MN975262.1,99.983,29781,5,0,1,29781,29839,59,0.0,54968
....
</code></pre></div></div>

<p>the workflow:</p>

<ul>
  <li>extracts accession numbers (the second column) for hit table</li>
  <li>downloads all corresponding FASTA files</li>
  <li>aligned them using <code class="highlighter-rouge">lastz</code></li>
  <li>selected all sequences that align over at least 75% of the reference</li>
  <li>uses these sequences to create multiple alignment and phylogenetic tree</li>
</ul>

<h3 id="outputs-5">Outputs</h3>

<ol>
  <li>A multiple alignment of sequences that align over at least 75% of the reference</li>
  <li>A maximum likelihood tree</li>
</ol>

<h3 id="history-and-workflow-4">History and workflow</h3>

<p>Galaxy workspace (history) containing the most current analysis can be imported from <a href="https://usegalaxy.org/u/aun1/h/ncov-comp">here</a>.</p>

<p>The <a href="https://usegalaxy.org/u/aun1/w/comp">workflow</a> is available at Galaxy public site and can downloaded and installed on any Galaxy instance. It contains version information for all tools used in this analysis.</p>

<p>the workflow performs the following steps:</p>

<ul>
  <li>extracts accession numbers (the second column) for hit table</li>
  <li>downloads all corresponding FASTA files</li>
  <li>aligned them using <code class="highlighter-rouge">lastz</code></li>
  <li>selected all sequences that align over at least 75% of the reference</li>
  <li>uses these sequences to create multiple alignment and phylogenetic tree</li>
</ul>

<p><img src="/assets/media/sars/comp_wf.png" alt="" height="50%" width="50%" /></p>

<h1 id="training">Training</h1>

<p>We are passionate about training. So we are working in close collaboration with the <a href="https://galaxyproject.org/teach/gtn/" target="_blank">Galaxy Training Network (GTN)</a> to develop training materials of data analyses based on Galaxy <a class="citation" href="#batut2017community">(Batut <i>et al.</i>, 2017)</a>. These materials hosted on the GTN GitHub repository are available online at <a href="https://training.galaxyproject.org" target="_blank">https://training.galaxyproject.org</a>.</p>

<p>Want to learn more about RNA analyses? Take one of our guided tour or check out the following hands-on tutorials. We developed several tutorials and the remaining are from the GTN community (marked with <i class="fa fa-users" aria-hidden="true"></i>)</p>

<table class="table table-striped">
  <thead>
    <tr>
      <th>Lesson</th>
      <th>Slides</th>
      <th>Hands-on</th>
      <th>Input dataset</th>
      <th>Workflows</th>
      <th>Galaxy tour</th>
      <th>Galaxy History</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Introduction to Transcriptomics</td>
      <td><a href="https://galaxyproject.github.io/training-material/topics/transcriptomics/slides/introduction.html" target="_blank"><i class="fa fa-slideshare" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td>¬†</td>
      <td>¬†</td>
      <td>¬†</td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>RNA-seq counts to genes</td>
      <td>¬†</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-counts-to-genes/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://figshare.com/s/1d788fd384d33e913a2a" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td>¬†</td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>RNA-seq genes to pathways</td>
      <td>¬†</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-genes-to-pathways/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/2596382" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td>¬†</td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>RNA-Seq reads to counts</td>
      <td>¬†</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-reads-to-counts/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://figshare.com/s/f5d63d8c265a05618137" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td>¬†</td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>Analyse unaligned ncRNAs</td>
      <td>¬†</td>
      <td>¬†</td>
      <td>¬†</td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=5cd167ed9e159e73" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>CLIP-Seq data analysis from pre-processing to motif detection <i class="fa fa-users" aria-hidden="true"></i></td>
      <td>¬†</td>
      <td><a href="https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/clipseq/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/1327423" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=f5be5bcf9b9f171c" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td><a href="https://virology.usegalaxy.eu/u/joerg-fallmann/h/eclipworkflow" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>De novo transcriptome reconstruction with RNA-Seq <i class="fa fa-users" aria-hidden="true"></i></td>
      <td>¬†</td>
      <td><a href="https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/de-novo/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/254485" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=f026c4b8341ff94c" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/tours/rnateam.de-novo" target="_blank"><i class="fa fa-magic" aria-hidden="true"></i></a></td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>Differential abundance testing of small RNAs <i class="fa fa-users" aria-hidden="true"></i></td>
      <td>¬†</td>
      <td><a href="https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/srna/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/826906" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=7734928ebc0a2654" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a> <a href="https://virology.usegalaxy.eu/workflows/run?id=1ffc058273ab357e" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/tours/differential_abundance_testing_sRNAs" target="_blank"><i class="fa fa-magic" aria-hidden="true"></i></a></td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>Network analysis with Heinz <i class="fa fa-users" aria-hidden="true"></i></td>
      <td><a href="https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html" target="_blank"><i class="fa fa-slideshare" aria-hidden="true"></i></a></td>
      <td><a href="https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/network-analysis-with-heinz/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/1344105" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=12c80c5b5e2305d8" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/tours/rnateam.network-analysis-with-heinz" target="_blank"><i class="fa fa-magic" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/u/videmp/h/rna-workbench-network-analysis-with-heinz" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>PAR-CLIP analysis</td>
      <td>¬†</td>
      <td>¬†</td>
      <td><a href="http://doi.org/10.5281/zenodo.2553519" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=a108b575b16e6cb9" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td><a href="https://virology.usegalaxy.eu/u/joerg-fallmann/h/parclipworkflow" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>Reference-based RNA-Seq data analysis</td>
      <td>¬†</td>
      <td><a href="https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/ref-based/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/1185122" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=9c7a218993788493" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/tours/ref_based_rna-seq" target="_blank"><i class="fa fa-magic" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/u/andrea.bagnacani/h/reference-based-rna-seq" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>RNA family model construction</td>
      <td>¬†</td>
      <td>¬†</td>
      <td>¬†</td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=8f2d958cee428ca1" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>RNA-seq counts to genes</td>
      <td>¬†</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-counts-to-genes/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://figshare.com/s/f5d63d8c265a05618137" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=86f89f49431b1e2e" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td><a href="https://virology.usegalaxy.eu/u/videmp/h/rna-workbench-rna-seq-counts-to-genes" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>RNA-seq genes to pathways</td>
      <td>¬†</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-genes-to-pathways/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/2596382" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=3cb45f0d38e9fd42" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td><a href="https://virology.usegalaxy.eu/u/videmp/h/rna-workbench-rna-seq-genes-to-pathways" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>RNA-Seq reads to counts</td>
      <td>¬†</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-reads-to-counts/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://figshare.com/s/f5d63d8c265a05618137" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=e89761c4bb25d89c" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td><a href="https://virology.usegalaxy.eu/u/videmp/h/rna-workbench-rna-seq-reads-to-counts-1" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>Scan for C/D-box sequences with segmentation-fold</td>
      <td>¬†</td>
      <td>¬†</td>
      <td>¬†</td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=3b717623054d5125" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>Small Non-coding RNA Clustering using BlockClust</td>
      <td>¬†</td>
      <td><a href="https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/small_ncrna_clustering/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/1491876" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=c7026cd5578c8678" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td><a href="https://virology.usegalaxy.eu/u/videmp/h/rna-workbench-small-non-coding-rna-clustering-using-blockclust" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>Visualization of RNA-Seq results with CummeRbund</td>
      <td><a href="https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/rna-seq-viz-with-cummerbund/slides.html" target="_blank"><i class="fa fa-slideshare" aria-hidden="true"></i></a></td>
      <td><a href="https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/rna-seq-viz-with-cummerbund/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/1001880" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=17e720bee3b9104f" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/tours/rna-seq-viz-with-cummerbund" target="_blank"><i class="fa fa-magic" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/u/videmp/h/rna-workbench-visualization-of-rna-seq-results-with-cummerbund" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>Visualization of RNA-Seq results with Volcano Plot</td>
      <td>¬†</td>
      <td><a href="https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/rna-seq-viz-with-volcanoplot/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/2529117" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=fd156028b09d213a" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td><a href="https://virology.usegalaxy.eu/u/videmp/h/rna-workbench-visualization-of-rna-seq-results-with-volcano-plot" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>Visualization of RNA-Seq results with heatmap2</td>
      <td>¬†</td>
      <td><a href="https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/rna-seq-viz-with-heatmap2/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/2529926" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=4dae6d48ba08c037" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td><a href="https://rna.usegalaxy.eu/u/videmp/h/rna-workbench-visualization-of-rna-seq-results-with-heatmap2" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>ViennaRNA Introduction</td>
      <td>¬†</td>
      <td>¬†</td>
      <td><a href="http://doi.org/10.5281/zenodo.2555028" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/workflows/run?id=58fd339165ded462" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/tours/rnateam.viennarna" target="_blank"><i class="fa fa-magic" aria-hidden="true"></i></a></td>
      <td><a href="https://virology.usegalaxy.eu/u/joerg-fallmann/h/viennarnaintroduction" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
  </tbody>
</table>

<h1 id="available-tools">Available tools</h1>

<p>In this section we list all tools that have been integrated in the RNA workbench. The list is likely to grow as soon as further tools and workflows are contributed. To ease readability, we divided them into categories.</p>

<h2 id="rna-structure-prediction-and-analysis">RNA structure prediction and analysis</h2>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th>Tool</th>
      <th>Description</th>
      <th>Reference</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/antarna/antarna" target="_top" title="antaRNA">antaRNA</a></td>
      <td>Possibility of inverse RNA structure folding and a specification of a GC value constraint</td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btv319" target="_blank">Kleinkauf et al. 2015</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/cofold/cofold" target="_top" title="CoFold">CoFold</a></td>
      <td>A thermodynamics-based RNA secondary structure folding algorithm</td>
      <td><a href="https://doi.org/10.1093/nar/gkt174" target="_blank">Proctor and Meyer, 2015</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/kinwalker/rbc_kinwalker" target="_top" title="Kinwalker">Kinwalker</a></td>
      <td>Algorithm for cotranscriptional folding of RNAs to obtain the min. free energy structure</td>
      <td><a href="https://doi.org/10.1016/j.jmb.2008.02.064" target="_blank">Geis et al. 2008</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/mea/mea" target="_top" title="MEA">MEA</a></td>
      <td>Prediction of maximum expected accuracy RNA secondary structures</td>
      <td><a href="https://doi.org/10.1007/978-3-319-02624-4_1" target="_blank">Amman et al. 2013</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/rnashapes/RNAshapes" target="_top" title="RNAshapes">RNAshapes</a></td>
      <td>Structures to a tree-like domain of shapes, retaining adjacency and nesting of structural features</td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btu649" target="_blank">Janssen and Giergerich, 2014</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/rnaz/rnaz" target="_top" title="RNAz">RNAz</a></td>
      <td>Predicts structurally conserved and therm. stable RNA secondary structures in mult. seq. alignments</td>
      <td><a href="https://doi.org/10.1073/pnas.0409169102" target="_blank">Washietl et al. 2005</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/yhoogstrate/segmentation_fold/segmentation_fold" target="_top" title="segmentation-fold">segmentation-fold</a></td>
      <td>An application that predicts RNA 2D-structure with an extended version of the Zuker algorithm</td>
      <td>-</td>
    </tr>
    <tr>
      <td>ViennaRNA</td>
      <td>A tool compilation for prediction and comparison of RNA secondary structures</td>
      <td><a href="https://doi.org/10.1186/1748-7188-6-26" target="_blank">Lorenz et al. 2011</a></td>
    </tr>
  </tbody>
</table>

<h2 id="rna-alignment">RNA alignment</h2>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th>Tool</th>
      <th>Description</th>
      <th>Reference</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/compalignp/compalignp" target="_top" title="Compalignp">Compalignp</a></td>
      <td>An RNA counterpart of the protein specific ‚ÄúBenchmark Alignment Database‚Äù</td>
      <td><a href="https://doi.org/10.1186/1748-7188-1-19" target="_blank">Wilm et al. 2006</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/mlocarna/mlocarna" target="_top" title="LocARNA">LocARNA</a></td>
      <td>A tool for multiple alignment of RNA molecules</td>
      <td><a href="https://doi.org/10.1261/rna.029041.111" target="_blank">Will et al. 2012</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/mafft/rbc_mafft" target="_top" title="MAFFT">MAFFT</a></td>
      <td>A multiple sequence alignment program for unix-like operating systems</td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btw108" target="_blank">Katoh and Standley, 2016</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/rnalien/RNAlien" target="_top" title="RNAlien">RNAlien</a></td>
      <td>A tool for RNA family model construction</td>
      <td><a href="https://doi.org/10.1093/nar/gkw558" target="_blank">Eggenhofer et al. 2016</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/cmv/cmv" target="_top" title="CMV">CMV</a></td>
      <td>RNA family model visualisation</td>
      <td><a href="https://doi.org/10.1093/bioinformatics/bty158" target="_blank">Eggenhofer et al. 2018</a></td>
    </tr>
  </tbody>
</table>

<h2 id="rna-annotation">RNA annotation</h2>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th>Tool</th>
      <th>Description</th>
      <th>Reference</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/trna_prediction/aragorn_trna" target="_top" title="ARAGORN">ARAGORN</a></td>
      <td>A tool to identify tRNA and tmRNA genes</td>
      <td><a href="https://doi.org/10.1093/nar/gkh152" target="_blank">Laslett and Canback, 2004</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/yhoogstrate/fuma/fuma" target="_top" title="Fusion Matcher (FuMa)">Fusion Matcher (FuMa)</a></td>
      <td>A tool that reports identical fusion genes based on gene-name annotations</td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btv721" target="_blank">Hoogstrate et al. 2016</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/gotohscan/rbc_gotohscan" target="_top" title="GotohScan">GotohScan</a></td>
      <td>A search tool that finds shorter sequences in large database sequences</td>
      <td><a href="https://doi.org/10.1093/nar/gkn1084" target="_blank">Hertel et al. 2009</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/infernal/infernal_cmsearch" target="_top" title="INFERNAL">INFERNAL</a></td>
      <td>A tool searching DNA sequence databases for RNA structure and sequence similarities</td>
      <td><a href="https://doi.org/10.1093/nar/gku1063" target="_blank">Nawrocki et al. 2015</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/rnabob/rbc_rnabob" target="_top" title="RNABOB">RNABOB</a></td>
      <td>A tool for fast pattern searching for RNA secondary structures</td>
      <td>-</td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/rnacode/rbc_rnacode" target="_top" title="RNAcode">RNAcode</a></td>
      <td>Predicts protein coding regions in a a set of homologous nucleotide sequences</td>
      <td><a href="https://doi.org/10.1261/rna.2536111" target="_blank">Washietl et al. 2011</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/trna_prediction/trnascan" target="_top" title="tRNAscan">tRNAscan</a></td>
      <td>Searches for tRNA genes in genomic sequences</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC146525/" target="_blank">Lowe and Eddy, 1997</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/rcas/rcas" target="_top" title="RCAS">RCAS</a></td>
      <td>A generic reporting tool for the functional analysis of transcriptome-wide regions of interest detected by high-throughput experiments</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28334930" target="_blank">Uyar et al.</a></td>
    </tr>
  </tbody>
</table>

<h2 id="rna-protein-interaction">RNA-protein interaction</h2>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th>Tool</th>
      <th>Description</th>
      <th>Reference</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/aresite2/AREsite2_REST" target="_top" title="AREsite2">AREsite2</a></td>
      <td>A database for AU-/GU-/U-rich elements in human and model organisms</td>
      <td><a href="https://doi.org/10.1093/nar/gkv1238" target="_blank">Fallmann et al. 2016</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/dorina/dorina_search" target="_top" title="DoRiNA">DoRiNA</a></td>
      <td>A database of RNA interactions in post-transcriptional regulation</td>
      <td><a href="https://doi.org/10.1093/nar/gku1180" target="_blank">Blin et al. 2014</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/paralyzer/paralyzer" target="_top" title="PARalyzer">PARalyzer</a></td>
      <td>An algorithm to generate a map of interacting RNA-binding proteins and their targets</td>
      <td><a href="https://doi.org/10.1186/gb-2011-12-8-r79" target="_blank">Corcoran et al. 2011</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/piranha/piranha" target="_top" title="Piranha">Piranha</a></td>
      <td>A peak-caller for CLIP- and RIP-seq data</td>
      <td>-</td>
    </tr>
  </tbody>
</table>

<h2 id="rna-rna-interaction">RNA-RNA interaction</h2>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th>Tool</th>
      <th>Description</th>
      <th>Reference</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>Unknown Tool</b><b>Unknown Tool</b><b>Unknown Tool</b><b>Unknown Tool</b><b>Unknown Tool</b></td>
      <td>A set of tools to analyze RNA-RNA interactome experimental data such as CLASH, CLEAR-CLIP, PARIS, LIGR-Seq etc</td>
      <td>-</td>
    </tr>
  </tbody>
</table>

<h2 id="rna-target-prediction">RNA target prediction</h2>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th>Tool</th>
      <th>Description</th>
      <th>Reference</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/targetfinder/targetfinder" target="_top" title="TargetFinder">TargetFinder</a></td>
      <td>A tool to predict small RNA binding sites on target transcripts from a sequence database</td>
      <td>-</td>
    </tr>
  </tbody>
</table>

<h2 id="rna-seq-and-hts-analysis">RNA Seq and HTS analysis</h2>

<h3 id="preprocessing">Preprocessing</h3>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th>Tool</th>
      <th>Description</th>
      <th>Reference</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/fastqc/fastqc" target="_top" title="FastQC">FastQC</a></td>
      <td>A quality control tool for high throughput sequence data</td>
      <td>-</td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/trim_galore/trim_galore" target="_top" title="Trim Galore!">Trim Galore!</a></td>
      <td>Automatic quality and adapter trimming as well as quality control</td>
      <td>-</td>
    </tr>
  </tbody>
</table>

<h3 id="rna-seq">RNA-Seq</h3>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th>Tool</th>
      <th>Description</th>
      <th>Reference</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/blockclust/blockclust" target="_top" title="BlockClust">BlockClust</a></td>
      <td>Small non-coding RNA clustering from deep sequencing read profiles</td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btu270" target="_blank">Videm et al. 2014</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/yhoogstrate/flaimapper/flaimapper" target="_top" title="FlaiMapper">FlaiMapper</a></td>
      <td>A tool for computational annotation of small ncRNA-derived fragments using RNA-seq data</td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btu696" target="_blank">Hoogstrate et al. 2015</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/mirdeep2/rbc_mirdeep2" target="_top" title="MiRDeep2">MiRDeep2</a></td>
      <td>Discovers microRNA genes by analyzing sequenced RNAs</td>
      <td><a href="https://doi.org/10.1038/nbt1394" target="_blank">Friedl√§nder et al. 2008</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/nastiseq/nastiseq" target="_top" title="NASTIseq">NASTIseq</a></td>
      <td>A method that incorporates the inherent variable efficiency of generating perfectly strand-specific libraries</td>
      <td><a href="https://doi.org/10.1101/gr.149310.112" target="_blank">Li et al. 2013</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/pipmir/pipmir" target="_top" title="PIPmiR">PIPmiR</a></td>
      <td>An algorithm to identify novel plant miRNA genes from a combination of deep sequencing data and genomic features</td>
      <td><a href="https://doi.org/10.1101/gr.123547.111" target="_blank">Breakfield et al. 2011</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/sortmerna/bg_sortmerna" target="_top" title="SortMeRNA">SortMeRNA</a></td>
      <td>A tool for filtering, mapping and OTU-picking NGS reads in metatranscriptomic and -genomic data</td>
      <td><a href="https://doi.org/10.1093/bioinformatics/bts611" target="_blank">Kopylova et al. 2011</a></td>
    </tr>
  </tbody>
</table>

<h3 id="read-mapping">Read Mapping</h3>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th>Tool</th>
      <th>Description</th>
      <th>Reference</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/hisat2/hisat2" target="_top" title="HISAT2">HISAT2</a></td>
      <td>Hierarchical indexing for spliced alignment of transcripts</td>
      <td><a href="https://doi.org/10.1038/nprot.2016.095" target="_blank">Pertea et al. 2016</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star" target="_top" title="RNA STAR">RNA STAR</a></td>
      <td>Rapid spliced aligner for RNA-seq data</td>
      <td><a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537/STAR-ultrafast-universal-RNA-seq-aligner" target="_blank">Dobin et al. 2013</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/star_fusion/star_fusion" target="_top" title="STAR-fusion">STAR-fusion</a></td>
      <td>Fast fusion gene finder</td>
      <td><a href="https://www.biorxiv.org/content/early/2017/03/24/120295" target="_blank">Haas et al. 2017</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/bowtie2/bowtie2" target="_top" title="Bowtie2">Bowtie2</a></td>
      <td>Fast and sensitive read alignment</td>
      <td><a href="https://doi.org/10.1038/nmeth.1923" target="_blank">Langmead et al. 2012</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/bwa/bwa" target="_top" title="BWA">BWA</a></td>
      <td>Software package for mapping low-divergent sequences against a large reference genome</td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank">Li and Durbin 2009</a>, <a href="https://doi.org/10.1093/bioinformatics/btp698" target="_blank">Li and Durbin 2010</a></td>
    </tr>
  </tbody>
</table>

<h3 id="transcript-assembly">Transcript Assembly</h3>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th>Tool</th>
      <th>Description</th>
      <th>Reference</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/trinity/trinity" target="_top" title="Trinity">Trinity</a></td>
      <td>De novo transcript sequence reconstruction from RNA-Seq</td>
      <td><a href="https://doi.org/10.1038%2Fnprot.2013.084" target="_blank">Haas et al. 2013</a></td>
    </tr>
  </tbody>
</table>

<h3 id="quantification">Quantification</h3>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th>Tool</th>
      <th>Description</th>
      <th>Reference</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/featurecounts/featurecounts" target="_top" title="featureCounts">featureCounts</a></td>
      <td>Ultrafast and accurate read summarization program</td>
      <td><a href="http://dx.doi.org/10.1093/bioinformatics/btt656" target="_blank">Liao et al. 2014</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=https://toolshed.g2.bx.psu.edu/repos/lparsons/htseq_count/" target="_top" title="htseq-count">htseq-count</a></td>
      <td>Tool for counting reads in features</td>
      <td><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu638" target="_blank">Anders et al. 2015</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/sailfish/sailfish" target="_top" title="Sailfish">Sailfish</a></td>
      <td>Rapid Alignment-free Quantification of Isoform Abundance</td>
      <td><a href="http://dx.doi.org/10.1038/nbt.2862" target="_blank">Patro et al. 2014</a></td>
    </tr>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/salmon/salmon" target="_top" title="Salmon">Salmon</a></td>
      <td>Fast, accurate and bias-aware transcript quantification</td>
      <td><a href="http://dx.doi.org/10.1038/nmeth.4197" target="_blank">Patro et al. 2017</a></td>
    </tr>
  </tbody>
</table>

<h3 id="differential-expression-analysis">Differential expression analysis</h3>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th>Tool</th>
      <th>Description</th>
      <th>Reference</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><a href="https://rna.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/deseq2/deseq2" target="_top" title="DESeq2">DESeq2</a></td>
      <td>Differential gene expression analysis based on the negative binomial distribution</td>
      <td><a href="http://doi.org/10.1186/s13059-014-0550-8" target="_blank">Love et al. 2014</a></td>
    </tr>
  </tbody>
</table>

<h3 id="utilities">Utilities</h3>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th>Tool</th>
      <th>Description</th>
      <th>Reference</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>SAMtools</td>
      <td>Utilities for manipulating alignments in the SAM format</td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btp352" target="_blank">Heng et al. 2009</a></td>
    </tr>
    <tr>
      <td>BEDTools</td>
      <td>Utilities for genome arithmetic</td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btq033" target="_blank">Quinlan and Hall 2010</a></td>
    </tr>
    <tr>
      <td>deepTools</td>
      <td>Tools for exploring deep-sequencing data</td>
      <td><a href="https://doi.org/10.1093/nar/gku365" target="_blank">Ramirez et al. 2014</a>, <a href="https://doi.org/10.1093/nar/gkw257" target="_blank">Ramirez et al. 2016</a></td>
    </tr>
  </tbody>
</table>

<h2 id="ribosome-profiling">Ribosome profiling</h2>

<table class="table table-striped tooltable">
  <thead>
    <tr>
      <th>Tool</th>
      <th>Description</th>
      <th>Reference</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>RiboTaper</td>
      <td>An analysis pipeline for Ribo-Seq experiments, exploiting the triplet periodicity of ribosomal footprints to call translated regions</td>
      <td><a href="https://doi.org/10.1038/nmeth.3688" target="_blank">Calviello et al. 2016</a></td>
    </tr>
  </tbody>
</table>

<h1 id="contributors">Contributors</h1>

<p><a href="https://github.com/dannon">Dannon Baker</a>,
<a href="https://github.com/mvdbeek">Marius Van Den Beek</a>, 
<a href="https://github.com/jmchilton">John Chilton</a>, 
<a href="https://github.com/natefoo">Nate Coraor</a>, 
<a href="https://github.com/bgruening">Bjorn Gruning</a>,
<a href="https://github.com/Delphine-L">Delphine Larivi√®re</a>, 
<a href="https://github.com/nickeener">Nicholas Keener</a>, 
<a href="https://github.com/spond">Sergei Kosakovsky</a>, 
<a href="https://github.com/wm75">Wolfgang Maier</a>,
<a href="https://github.com/nekrut">Anton Nekrutenko</a>, 
<a href="https://github.com/jxtx">James Taylor</a>, 
<a href="https://github.com/stevenweaver">Steven Weaver</a></p>


<h2>Our Data Policy</h2>


  <table class="table table-striped">
    <thead>
      <tr>
        <th>Registered Users</th><th>Unregistered Users</th><th>FTP Data</th><th>GDPR Compliance</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>User data on UseGalaxy.eu (i.e. datasets, histories) will be available as long
            as they are not deleted by the user. Once marked as deleted the datasets will
            be permanently removed within 14 days. If the user "purges" the dataset in the
            Galaxy, it will be removed immediately, permanently.

            An <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9w2MOS6KOlu9XdhRSDqWnCDkzoVBqHJ3zH_My4p8D8ZgkIQ/viewform" target="_blank">extended quota can be requested</a>
for a limited time period in special cases.
        </td>
        <td>Processed data will only be accessible during one browser session, using a
            cookie to identify your data. This cookie is not used for any other purposes
            (e.g. tracking or analytics).
            
            If UseGalaxy.eu service is not accessed for 90 days, those datasets will be
            permanently deleted.
        </td>
        <td>Any user data uploaded to our <a href="https://galaxyproject.eu/ftp/">FTP server</a> should be imported into Galaxy as soon
            as possible. Data left in FTP folders for more than 3 months, will be deleted.            
        </td>
        <td>The Galaxy service complies with the EU General Data Protection Regulation
            (GDPR). You can read more about this on our
            <a href="https://usegalaxy.eu/terms/">Terms and Conditions</a>.</td>
      </tr>
    </tbody>
  </table>



<!-- <h4>Registered Users</h4>

User data on UseGalaxy.eu (i.e. datasets, histories) will be available as long
as they are not deleted by the user. Once marked as deleted the datasets will
be permanently removed within 14 days. If the user "purges" the dataset in the
Galaxy, it will be removed immediately, permanently.

An <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9w2MOS6KOlu9XdhRSDqWnCDkzoVBqHJ3zH_My4p8D8ZgkIQ/viewform" target="_blank">extended quota can be requested</a>
for a limited time period in special cases.

<h4>Unregistered Users</h4>

Processed data will only be accessible during one browser session, using a
cookie to identify your data. This cookie is not used for any other purposes
(e.g. tracking or analytics).

If UseGalaxy.eu service is not accessed for 90 days, those datasets will be
permanently deleted.

<h4>FTP Data</h4>

Any user data uploaded to our <a href="https://galaxyproject.eu/ftp/">FTP server</a> should be imported into Galaxy as soon
as possible. Data left in FTP folders for more than 3 months, will be deleted.

<h4>GDPR Compliance</h4>

The Galaxy service complies with the EU General Data Protection Regulation
(GDPR). You can read more about this on our
<a href="https://usegalaxy.eu/terms/">Terms and Conditions</a>. 
 -->

<div>
    <iframe style="border: 0px" width="100%" height="150px" src="https://stats.galaxyproject.eu/d-solo/000000034/jobs-dashboard?orgId=1&refresh=1m&panelId=1" ></iframe>
</div>
<div>
<!--<iframe style="border: 0px" width="33%" height="100px" src="https://stats.galaxyproject.eu/d-solo/000000034/jobs-dashboard?orgId=1&refresh=1m&panelId=3" ></iframe>-->
</div>

<div class="row">

    <section class="section-content">
        <div class="col-md-12">
        </div>
    </section>
</div>

</div>

        </div>
      </div>
    </div>
    







<footer class="navbar-default">
    <div class="container">
        <div class="row">
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <p>UseGalaxy.eu is maintained largely by the <a href="/freiburg/">Freiburg Galaxy Team</a> but also collectively by groups and individuals from across Europe. All of the member sites in this repository contribute to the European Galaxy Project.
For <strong>acknowledgement</strong>, please refer to the <a href="/about">About</a> section.
All content on this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a> unless otherwise specified.</p>

            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu/website/tree/master/index-virology.md">Edit this page on GitHub</a></li>
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:contact@usegalaxy.eu">contact@usegalaxy.eu</a></li>
                    
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu">usegalaxy-eu</a></li>
                    
                    
                      <li><i class="fa fa-twitter"></i><a href="https://twitter.com/galaxyproject">galaxyproject</a></li>
                    
                      <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li>
                </ul>
            </div>
        </div>
    </div>
</footer>

<script async defer data-domain="galaxyproject.eu" src="https://plausible.galaxyproject.eu/js/plausible.js"></script>
  </body>
</html>
