<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy Europe</title>
  <meta property="og:title" content="" />
  <meta property="og:description" content="" />
  <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" />
  <meta name="description" content="The European Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://galaxyproject.eu/index-imaging.html">
  <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy Europe" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>

  <body>
    <div id="wrap">
      <div id="main">
        <div class="container" id="maincontainer">
          <div class="home">










<div class="alert alert-success" style="margin-top: 0.5em">
	<h4>COVID-19 Research!</h4>
	
	
	<div><p>Want to learn the best practices for the analysis of SARS-CoV-2 data using Galaxy?
Visit the <a href="https://covid19.galaxyproject.org">Galaxy SARS-CoV-2 portal</a>.
We mirror <b>all public SARS-CoV-2 data</b> from <a href="https://www.ebi.ac.uk/ena/browser/home">ENA</a> in a <a href="https://bit.ly/usegalaxy-eu-covid19-data">Galaxy data library</a> for your convenience.
The Galaxy community has created <a href="https://training.galaxyproject.org/search?query=covid19">COVID-19 dedicated training materials</a>.
Please check our <a href="https://docs.google.com/document/d/1Ajfy3RG-BOGTSdhQUMZ0hnGtePZplv-lK0rYzx8L-vo/preview#">recent activities</a> for more details.</p>

<p>If you need help submitting your data to public archives, like ENA, please <a href="mailto:contact@usegalaxy.eu">get in touch</a>. We will support you in sharing your data.</p>
</div>
	
</div>








<div class="alert alert-info" style="margin-top: 0.5em">
	<h4>Holiday season</h4>
	
	
	<div><p>Galaxy Europe will continue to operate over the holiday season although with reduced support capacity. This period will run from 24th December 2021 until the 4th of January 2022.
During this time:
    No new tools or reference data will be installed
    No requests for disk space will be completed
    No support tickets will be responded to
    No TIaaS requests will be evaluated</p>

<p>We will also have limited service administration capacity and while we will make all reasonable efforts to keep the service running, there may be issues outside of our control during this period.</p>

<p>The Galaxy Europe team wish all of our users a safe, relaxing and happy holiday season.</p>
</div>
	
</div>





  <h1 class="no_toc" id="welcome-to-the-imaging-flavour-of-galaxy">Welcome to the Imaging flavour of Galaxy</h1>

<p>The imaging flavor of Galaxy offers a set of training materials and tools that you can assemble into workflows to perform image analysis.</p>

<ol id="markdown-toc">
  <li><a href="#get-started" id="markdown-toc-get-started">Get started</a></li>
  <li><a href="#training" id="markdown-toc-training">Training</a>    <ol>
      <li><a href="#introduction-to-galaxy--image-analysis" id="markdown-toc-introduction-to-galaxy--image-analysis">Introduction to Galaxy &amp; Image analysis</a></li>
      <li><a href="#segmentation--feature-extraction" id="markdown-toc-segmentation--feature-extraction">Segmentation &amp; Feature extraction</a></li>
      <li><a href="#mass-spectrometry-imaging" id="markdown-toc-mass-spectrometry-imaging">Mass Spectrometry Imaging</a></li>
    </ol>
  </li>
  <li><a href="#available-tools" id="markdown-toc-available-tools">Available tools</a>    <ol>
      <li><a href="#data-access" id="markdown-toc-data-access">Data access</a></li>
      <li><a href="#segmentation--feature-extraction-1" id="markdown-toc-segmentation--feature-extraction-1">Segmentation &amp; Feature extraction</a></li>
      <li><a href="#mass-spectrometry-imaging-1" id="markdown-toc-mass-spectrometry-imaging-1">Mass Spectrometry Imaging</a></li>
      <li><a href="#biological-oscillations-analysis" id="markdown-toc-biological-oscillations-analysis">Biological Oscillations Analysis</a></li>
    </ol>
  </li>
  <li><a href="#partners" id="markdown-toc-partners">Partners</a></li>
</ol>

<h1 id="get-started">Get started</h1>

<p>Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take <a href="https://imaging.usegalaxy.eu/tours/core.galaxy_ui" target="_blank">a guided tour</a> through Galaxy’s user interface.</p>

<p><br /></p>
<h1 id="training">Training</h1>

<p><a href="https://training.galaxyproject.org/training-material/topics/imaging/" target="_blank">Several image analysis tutorials</a> avalable in the <a href="https://training.galaxyproject.org" target="_blank"><strong>Galaxy Training Network (GTN)</strong></a>. If you want to know more about the GTN, check the video below!</p>

<p><br /></p>

<iframe width="560" height="315" src="https://www.youtube.com/embed/lDqWxzWNk1k" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
</iframe>

<p><br /></p>

<h2 id="introduction-to-galaxy--image-analysis">Introduction to Galaxy &amp; Image analysis</h2>

<table class="table table-striped">
  <thead>
    <tr>
      <th>Topic</th>
      <th>Slides</th>
      <th>Hands-on</th>
      <th>Input dataset</th>
      <th>Workflows</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Welcome and introduction to Galaxy</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-short/slides.html" target="_blank"><i class="fa fa-slideshare" aria-hidden="true"></i></a></td>
      <td> </td>
      <td> </td>
      <td> </td>
    </tr>
    <tr>
      <td>Introduction to image analysis using Galaxy</td>
      <td> </td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/imaging-introduction/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="http://doi.org/10.5281/zenodo.3362976" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/imaging-introduction/workflows/" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
    </tr>
  </tbody>
</table>

<h2 id="segmentation--feature-extraction">Segmentation &amp; Feature extraction</h2>

<table class="table table-striped">
  <thead>
    <tr>
      <th>Topic</th>
      <th>Slides</th>
      <th>Hands-on</th>
      <th>Input dataset</th>
      <th>Workflows</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Analyse HeLa fluorescence siRNA screen</td>
      <td> </td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/hela-screen-analysis/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="http://doi.org/10.5281/zenodo.3362976" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/hela-screen-analysis/workflows/" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>Nucleoli segmentation and feature extraction using CellProfiler</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/tutorial-CP/slides.html#1" target="_blank"><i class="fa fa-slideshare" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/tutorial-CP/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td> </td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/tutorial-CP/workflows/" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>Object tracking using CellProfiler</td>
      <td> </td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/object-tracking-using-cell-profiler/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="http://doi.org/10.5281/zenodo.4567084" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/object-tracking-using-cell-profiler/workflows/" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
    </tr>
  </tbody>
</table>

<h2 id="mass-spectrometry-imaging">Mass Spectrometry Imaging</h2>

<table class="table table-striped">
  <thead>
    <tr>
      <th>Topic</th>
      <th>Slides</th>
      <th>Hands-on</th>
      <th>Input dataset</th>
      <th>Workflows</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Mass spectrometry imaging: Loading and exploring MSI data</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/proteomics/slides/introduction.html#1" target="_blank"><i class="fa fa-slideshare" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/proteomics/tutorials/mass-spectrometry-imaging-loading-exploring-data/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="http://doi.org/10.5281/zenodo.1560646" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/proteomics/tutorials/mass-spectrometry-imaging-loading-exploring-data/workflows/" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>Mass spectrometry imaging: Examining the spatial distribution of analytes</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/metabolomics/slides/introduction.html#1" target="_blank"><i class="fa fa-slideshare" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/metabolomics/tutorials/msi-analyte-distribution/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="http://doi.org/10.5281/zenodo.484496" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/metabolomics/tutorials/msi-analyte-distribution/workflows/" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>Mass spectrometry imaging: Finding differential analytes</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/metabolomics/slides/introduction.html#1" target="_blank"><i class="fa fa-slideshare" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/metabolomics/tutorials/msi-analyte-distribution/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="http://doi.org/10.5281/zenodo.484496" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/metabolomics/tutorials/msi-analyte-distribution/workflows/" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
    </tr>
  </tbody>
</table>

<p><br /></p>
<h1 id="available-tools">Available tools</h1>

<h2 id="data-access">Data access</h2>

<p>To access your image data, you can always upload it to your history. However, since the images are usually heavy, there are connectors to existing public databases that will bring the data that you need into Galaxy.</p>

<table class="table table-striped">
  <thead>
    <tr>
      <th>TOOL</th>
      <th>DESCRIPTION</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong>IDR/OMERO Download Tool</strong></td>
      <td>Access to public images at the <a href="https://idr.openmicroscopy.org/">IDR</a>. This tool allows <a href="https://galaxyproject.eu/posts/2020/11/23/OMERO-post/">access to image data in private OMERO instances</a>.</td>
    </tr>
    <tr>
      <td><strong>Convert image</strong></td>
      <td>Tool to convert image formats.</td>
    </tr>
  </tbody>
</table>

<h2 id="segmentation--feature-extraction-1">Segmentation &amp; Feature extraction</h2>

<p>There are many tools organised into suites to segment and extract features from your images.</p>

<table class="table table-striped">
  <thead>
    <tr>
      <th>TOOL</th>
      <th>DESCRIPTION</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong>CellProfiler Suite</strong></td>
      <td>CellProfiler modules available: Starting Modules (Images, Metadata, NamesAndTypes, Groups), IdentifyPrimaryObjects, OverlayOutlines, ImageMath, TrackObjects, DisplayDataOnImage, ColorToGray, GrayToColor, Tile, ConvertObjectsToImage, RelateObjects, MeasureImageIntensity, MeasureObjectIntensity, MeasureGranularity, MeasureTexture, MeasureImageAreaOccupied, MeasureImageQuality, MaskImage, MeasureObjectSizeShape,  Export to Spreadsheet, EnhanceOrSuppressFeatures, SaveImages. <strong>Every workflow needs to start by <em>Starting Modules</em> and end by <em>CellProfiler</em></strong>. The last tool takes two inputs: an image collection and a CellProfiler pipeline (assembled in previous steps of the workflow).</td>
    </tr>
    <tr>
      <td><strong>ImageJ Suite</strong></td>
      <td>Several plugins are available in Galaxy: Add shadow effect, Analyze skeleton, Watershed segmentation, Adjust threshold, Convert binary image to EDM, Skeletonize, Analyze particles, Operate on pixels, Add or remove noise, Find maxima, Convert to binary, Enhance contrast, Create new image, Sharpen, Find edges, Smooth.</td>
    </tr>
    <tr>
      <td><strong>bUnwarpJ (ImageJ)</strong></td>
      <td>Apply raw transformation, Convert elastic transformation to raw, Apply elastic transformation, Compare elastic and raw deformation, Align two images, Compare opposite elastic deformations, Compose two raw transformations, Compare two raw deformations, Compose two elastic transformations, Compose a raw and an elastic transformation, Adapt an elastic transformation.</td>
    </tr>
  </tbody>
</table>

<h2 id="mass-spectrometry-imaging-1">Mass Spectrometry Imaging</h2>

<table class="table table-striped">
  <thead>
    <tr>
      <th>TOOL</th>
      <th>DESCRIPTION</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong>MALDIquant Suite</strong></td>
      <td>Pre-processing, peak detection, alignment and filtering of mass spectrometry imaging data.</td>
    </tr>
    <tr>
      <td><strong>Cardinal Suite</strong></td>
      <td>filtering, combining, raw data exporting, quality report, ion images, spectra plots, segmentation and classification of mass spectrometry imaging data.</td>
    </tr>
  </tbody>
</table>

<h2 id="biological-oscillations-analysis">Biological Oscillations Analysis</h2>

<table class="table table-striped">
  <thead>
    <tr>
      <th>TOOL</th>
      <th>DESCRIPTION</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong>SpyBOAT</strong></td>
      <td>Wavelet analysis of spatially extended oscillatory systems, represented as 3D-image stacks.</td>
    </tr>
  </tbody>
</table>

<p><br /><br /></p>

<h1 id="partners">Partners</h1>

<p>The imaging flavour of Galaxy is a joint effort between different partners including EOSC-Life and its associated Research Infrasctuctures (<a href="https://www.eurobioimaging.eu/">Euro-Bioimaging</a>, <a href="https://elixir-europe.org/">ELIXIR</a>), and the <a href="https://galaxyproject.eu/freiburg/" target="_blank">University of Freiburg</a>.</p>

<p>The service is part of the European Galaxy server and is maintained by the <a href="https://www.denbi.de/network/rna-bioinformatics-center-rbc" target="_blank">RNA Bioinformatics Center (RBC)</a> as part of <a href="https://www.denbi.de" target="_blank">de.NBI</a> and <a href="http://elixir-europe.org" target="_blank">ELIXIR</a>.</p>

<table border="0"><tr>
<td width="10%">
<img alt="EOSC-Life" src="/assets/media/EOSC_logo.png" />
</td>
<td with="8%"></td>
<td width="12%">
<img alt="Galaxy Freiburg" src="/assets/media/freiburg-galaxy.svg" />
</td>
<td with="0%"></td>
<td width="30%">
<img alt="Euro-Bioimaging" src="/assets/media/eubi_logo.png" />
</td>
<td with="0%"></td>
<td width="10%">
<img alt="ELIXIR" src="/assets/media/elixir_logo.png" />
</td>
<td with="1%"></td>
<td width="13%">
<img alt="EMBL" src="/assets/media/logo_EMBL.png" />
</td>
<td with="3%"></td>
<td width="10%">
<img alt="IDR" src="/assets/media/idr_logo.png" />
</td>
</tr></table>


<h2>Our Data Policy</h2>


  <table class="table table-striped">
    <thead>
      <tr>
        <th>Registered Users</th><th>Unregistered Users</th><th>FTP Data</th><th>GDPR Compliance</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>User data on UseGalaxy.eu (i.e. datasets, histories) will be available as long
            as they are not deleted by the user. Once marked as deleted the datasets will
            be permanently removed within 14 days. If the user "purges" the dataset in the
            Galaxy, it will be removed immediately, permanently.

            An <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9w2MOS6KOlu9XdhRSDqWnCDkzoVBqHJ3zH_My4p8D8ZgkIQ/viewform" target="_blank">extended quota can be requested</a>
for a limited time period in special cases.
        </td>
        <td>Processed data will only be accessible during one browser session, using a
            cookie to identify your data. This cookie is not used for any other purposes
            (e.g. tracking or analytics).
            
            If UseGalaxy.eu service is not accessed for 90 days, those datasets will be
            permanently deleted.
        </td>
        <td>Any user data uploaded to our <a href="https://galaxyproject.eu/ftp/">FTP server</a> should be imported into Galaxy as soon
            as possible. Data left in FTP folders for more than 3 months, will be deleted.            
        </td>
        <td>The Galaxy service complies with the EU General Data Protection Regulation
            (GDPR). You can read more about this on our
            <a href="https://usegalaxy.eu/terms/">Terms and Conditions</a>.</td>
      </tr>
    </tbody>
  </table>



<!-- <h4>Registered Users</h4>

User data on UseGalaxy.eu (i.e. datasets, histories) will be available as long
as they are not deleted by the user. Once marked as deleted the datasets will
be permanently removed within 14 days. If the user "purges" the dataset in the
Galaxy, it will be removed immediately, permanently.

An <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9w2MOS6KOlu9XdhRSDqWnCDkzoVBqHJ3zH_My4p8D8ZgkIQ/viewform" target="_blank">extended quota can be requested</a>
for a limited time period in special cases.

<h4>Unregistered Users</h4>

Processed data will only be accessible during one browser session, using a
cookie to identify your data. This cookie is not used for any other purposes
(e.g. tracking or analytics).

If UseGalaxy.eu service is not accessed for 90 days, those datasets will be
permanently deleted.

<h4>FTP Data</h4>

Any user data uploaded to our <a href="https://galaxyproject.eu/ftp/">FTP server</a> should be imported into Galaxy as soon
as possible. Data left in FTP folders for more than 3 months, will be deleted.

<h4>GDPR Compliance</h4>

The Galaxy service complies with the EU General Data Protection Regulation
(GDPR). You can read more about this on our
<a href="https://usegalaxy.eu/terms/">Terms and Conditions</a>. 
 -->

<div>
    <iframe style="border: 0px" width="100%" height="150px" src="https://stats.galaxyproject.eu/d-solo/000000034/jobs-dashboard?orgId=1&refresh=1m&panelId=1" ></iframe>
</div>
<div>
<!--<iframe style="border: 0px" width="33%" height="100px" src="https://stats.galaxyproject.eu/d-solo/000000034/jobs-dashboard?orgId=1&refresh=1m&panelId=3" ></iframe>-->
</div>

<div class="row">

    <section class="section-content">
        <div class="col-md-12">
        </div>
    </section>
</div>

</div>

        </div>
      </div>
    </div>
    






<script>
  ((window.gitter = {}).chat = {}).options = {
  room: 'usegalaxy-eu/Lobby'
  };
</script>
<script src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script>


<footer class="navbar-default">
    <div class="container">
        <div class="row">
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <p>UseGalaxy.eu is maintained largely by the <a href="/freiburg/">Freiburg Galaxy Team</a> but also collectively by groups and individuals from across Europe. All of the member sites in this repository contribute to the European Galaxy Project.
For <strong>acknowledgement</strong>, please refer to the <a href="/about">About</a> section.
All content on this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a> unless otherwise specified.</p>

            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu/website/tree/master/index-imaging.md">Edit this page on GitHub</a></li>
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:contact@usegalaxy.eu">contact@usegalaxy.eu</a></li>
                    
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu">usegalaxy-eu</a></li>
                    
                    
                      <li><i class="fa fa-twitter"></i><a href="https://twitter.com/galaxyproject">galaxyproject</a></li>
                    
                      <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li>
                </ul>
            </div>
        </div>
    </div>
</footer>

<script async defer data-domain="galaxyproject.eu" src="https://plausible.galaxyproject.eu/js/plausible.js"></script>
  </body>
</html>
