<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy Europe</title>
  <meta property="og:title" content="" />
  <meta property="og:description" content="" />
  <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" />
  <meta name="description" content="The European Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://galaxyproject.eu/index-microgalaxy.html">
  <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy Europe" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>

  <body>
    <div id="wrap">
      <div id="main">
        <div class="container" id="maincontainer">
          <div class="home">










<div class="alert alert-success" style="margin-top: 0.5em">
	<h4>COVID-19 Research!</h4>
	
	
	<div><p>Want to learn the best practices for the analysis of SARS-CoV-2 data using Galaxy?
Visit the <a href="https://covid19.galaxyproject.org">Galaxy SARS-CoV-2 portal</a>.
We mirror <b>all public SARS-CoV-2 data</b> from <a href="https://www.ebi.ac.uk/ena/browser/home">ENA</a> in a <a href="https://bit.ly/usegalaxy-eu-covid19-data">Galaxy data library</a> for your convenience.
The Galaxy community has created <a href="https://training.galaxyproject.org/search?query=covid19">COVID-19 dedicated training materials</a>.
Please check our <a href="https://docs.google.com/document/d/1Ajfy3RG-BOGTSdhQUMZ0hnGtePZplv-lK0rYzx8L-vo/preview#">recent activities</a> for more details.</p>

<p>If you need help submitting your data to public archives, like ENA, please <a href="mailto:contact@usegalaxy.eu">get in touch</a>. We will support you in sharing your data.</p>
</div>
	
</div>





  <p><img src="/assets/media/microgalaxy-logo.png" alt="microGalaxy logo" class="sc-intro-left" /></p>

<h1 id="welcome-to-microgalaxy">Welcome to microGalaxy!</h1>

<p class="no_toc"><br />
<strong>microGalaxy</strong> is the place to go for anything microbiology in Galaxy! Whether you are analysing microbiome samples or bacterial isolates, long reads or short, shotgun or 16S, genomics transcriptomics or proteomics, this is the place to be!</p>

<p><br /></p>
<h1 class="no_toc" id="content">Content</h1>

<ol id="markdown-toc">
  <li><a href="#welcome-to-microgalaxy" id="markdown-toc-welcome-to-microgalaxy">Welcome to microGalaxy!</a></li>
  <li><a href="#join-the-μgalaxy-community" id="markdown-toc-join-the-μgalaxy-community">Join the μGalaxy Community</a></li>
  <li><a href="#workflows" id="markdown-toc-workflows">Workflows</a>    <ol>
      <li><a href="#microbiome" id="markdown-toc-microbiome">Microbiome</a></li>
      <li><a href="#bacterial-isolates" id="markdown-toc-bacterial-isolates">Bacterial Isolates</a></li>
    </ol>
  </li>
  <li><a href="#tools" id="markdown-toc-tools">Tools</a></li>
  <li><a href="#projects" id="markdown-toc-projects">Projects</a></li>
  <li><a href="#resources" id="markdown-toc-resources">Resources</a></li>
</ol>

<h1 id="join-the-μgalaxy-community">Join the μGalaxy Community</h1>

<p>Anybody interested in microbiology in Galaxy is welcome to join our microGalaxy interest group! <strong>Everybody is Welcome!</strong></p>

<ul>
  <li>Join the Discussion! <a href="https://gitter.im/galaxyproject/microGalaxy">Gitter Chat</a> (also available via <a href="">Matrix</a>)</li>
  <li>Join our quarterly meetings! <a href="https://docs.google.com/document/d/13VjcUjStuIp7bK29e74k8Nqb7N4lmVcg1ioArEWr254/edit#">Agenda and minutes</a></li>
</ul>

<h1 id="workflows">Workflows</h1>

<p>Below are a list of curated Galaxy workflows for different kinds of microbial analysis. Many of these are accompanied by comprehensive <a href="https://training.galaxyproject.org">GTN Tutorials</a> that will guide you through the analysis step by step.</p>

<h2 id="microbiome">Microbiome</h2>

<table class="table table-striped">
  <thead>
    <tr>
      <th>Analysis</th>
      <th>Sequencing</th>
      <th>GTN Tutorial</th>
      <th>Workflow</th>
      <th>Notes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Taxonomic Profiling</td>
      <td>16S, short reads</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html">View Tutorial</a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/mothur-miseq-sop/workflows/">View Workflow</a></td>
      <td>mothur SOP</td>
    </tr>
    <tr>
      <td>Taxonomic Profiling</td>
      <td>16S, long reads</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.html">View Tutorial</a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/nanopore-16S-metagenomics/workflows/">View Workflow</a></td>
      <td>Nanopore</td>
    </tr>
    <tr>
      <td>Taxonomic Profiling, Functional Analysis</td>
      <td>Shotgun, short reads</td>
      <td> </td>
      <td><a href="https://asaim.readthedocs.io/en/latest/workflows.html">View Workflows</a></td>
      <td>ASaiM (missing reference)</td>
    </tr>
    <tr>
      <td>AMR detection</td>
      <td>Shotgun, plasmids, long reads</td>
      <td><a href="">View Tutorial</a></td>
      <td> </td>
      <td> </td>
    </tr>
    <tr>
      <td>Functional Analysis</td>
      <td>Transcriptomics</td>
      <td><a href="">View Tutorial</a></td>
      <td><a href="">View Workflow</a></td>
      <td>ASaiM-MT (missing reference)</td>
    </tr>
  </tbody>
</table>

<h2 id="bacterial-isolates">Bacterial Isolates</h2>

<table class="table table-striped">
  <thead>
    <tr>
      <th>Organism</th>
      <th>Analysis</th>
      <th>Sequencing</th>
      <th>GTN Tutorial</th>
      <th>Workflow</th>
      <th>Notes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>M. tuberculosis</td>
      <td>Variant Detection</td>
      <td>Short reads</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html">View Tutorial</a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/tb-variant-analysis/workflows/">View Workflow</a></td>
      <td> </td>
    </tr>
    <tr>
      <td>Plasmids</td>
      <td>AMR detection</td>
      <td>long reads, plasmids</td>
      <td> </td>
      <td><a href="https://erasmusmc-bioinformatics.github.io/AMR-Galaxy-workflows/">View Workflows</a></td>
      <td> </td>
    </tr>
  </tbody>
</table>

<p>Want to include your workflow here? Let us know!</p>

<h1 id="tools">Tools</h1>
<p>More than <strong>200 tools</strong> are avalaible for microbiome data analysis in this custom Galaxy instance:</p>

<ul>
  <li><strong>General tools</strong>
    <ul>
      <li><strong>Data retrieval</strong>: EBISearch, ENASearch, SRA Tools</li>
      <li><strong>BAM/SAM file manipulation</strong>: SAM tools</li>
      <li><strong>BIOM file manipulation</strong>: BIOM-Format tools</li>
    </ul>
  </li>
  <li><strong>Genomics tools</strong>
    <ul>
      <li><strong>Quality control</strong>: FastQC, PRINSEQ, Cutadapt, fastp, Trimmomatic, MultiQC</li>
      <li><strong>Clustering</strong>: CD-Hit</li>
      <li><strong>Sorting and prediction</strong>: SortMeRNA, FragGeneScan</li>
      <li><strong>Mapping</strong>: BWA, Bowtie</li>
      <li><strong>Similarity search</strong>: NCBI Blast+, Diamond</li>
      <li><strong>Alignment</strong>: HMMER3</li>
    </ul>
  </li>
  <li><strong>Microbiota dedicated tools</strong>
    <ul>
      <li><strong>Microbial</strong>: Scoary, Prokka, Roary</li>
      <li><strong>Metagenomics data manipulation</strong>: VSearch, Nonpareil, DADA2</li>
      <li><strong>Assembly</strong>: MEGAHIT, metaSPAdes, metaQUAST, VALET, Bandage, MaxBin2</li>
      <li><strong>Metataxonomic sequence analysis</strong>: Mothur, QIIME, Vegan</li>
      <li><strong>Taxonomy assignation</strong>: MetaPhlAn, Kraken, CAT/BAT</li>
      <li><strong>Metabolism assignation</strong>: HUMAnN, PICRUST, InterProScan</li>
      <li><strong>Visualization</strong>: Export2graphlan, GraPhlAn, KRONA</li>
      <li><strong>Metaproteomics</strong>: MaxQuant, SearchGUI, PeptideShaker, Unipept</li>
    </ul>
  </li>
</ul>

<h1 id="projects">Projects</h1>

<p>Below is a list of projects that members of this community are involved in (feel free to add your own!)</p>

<table class="table table-striped">
  <thead>
    <tr>
      <th>Project</th>
      <th>Description</th>
      <th>Links</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><img src="https://irida.ca/assets/images/IRIDA-logo-news.png" alt="" width="100px" /></td>
      <td>IRIDA: The Integrated Rapid Infectious Disease Analysis</td>
      <td><a href="https://irida.ca/">Website</a></td>
    </tr>
    <tr>
      <td><img src="https://www.chalmers.se/SiteCollectionImages/Institutioner/MV/Nyheter/Seq4AMRLogo200x.png" alt="" width="100px" /></td>
      <td>Seq4AMR: Fighting antibiotic resistance with new diagnostics</td>
      <td><a href="https://www.jpiamr.eu/projects/seq4amr/">Website</a></td>
    </tr>
  </tbody>
</table>

<p>If you would like to know more about any of these projects or get involved, please contact us know on the <a href="https://gitter.im/galaxyproject/microGalaxy">microGalaxy Gitter channel</a>.</p>

<h1 id="resources">Resources</h1>

<table class="table table-striped">
  <thead>
    <tr>
      <th>Resource</th>
      <th>Description</th>
      <th>Links</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><img src="https://www.jpiamr.eu/app/uploads/2021/09/JPIAMR-logo-no-tagline.png" alt="" width="100px" /></td>
      <td>JPIAMR: Global Coordination of Antimicrobial Resistance Research</td>
      <td><a href="https://www.jpiamr.eu/">Website</a></td>
    </tr>
  </tbody>
</table>

<!--
# Acknowledgements

# References

<ol class="bibliography"></ol>
-->


<h2>Our Data Policy</h2>


  <table class="table table-striped">
    <thead>
      <tr>
        <th>Registered Users</th><th>Unregistered Users</th><th>FTP Data</th><th>GDPR Compliance</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>User data on UseGalaxy.eu (i.e. datasets, histories) will be available as long
            as they are not deleted by the user. Once marked as deleted the datasets will
            be permanently removed within 14 days. If the user "purges" the dataset in the
            Galaxy, it will be removed immediately, permanently.

            An <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9w2MOS6KOlu9XdhRSDqWnCDkzoVBqHJ3zH_My4p8D8ZgkIQ/viewform" target="_blank">extended quota can be requested</a>
for a limited time period in special cases.
        </td>
        <td>Processed data will only be accessible during one browser session, using a
            cookie to identify your data. This cookie is not used for any other purposes
            (e.g. tracking or analytics).
            
            If UseGalaxy.eu service is not accessed for 90 days, those datasets will be
            permanently deleted.
        </td>
        <td>Any user data uploaded to our <a href="https://galaxyproject.eu/ftp/">FTP server</a> should be imported into Galaxy as soon
            as possible. Data left in FTP folders for more than 3 months, will be deleted.            
        </td>
        <td>The Galaxy service complies with the EU General Data Protection Regulation
            (GDPR). You can read more about this on our
            <a href="https://usegalaxy.eu/terms/">Terms and Conditions</a>.</td>
      </tr>
    </tbody>
  </table>



<!-- <h4>Registered Users</h4>

User data on UseGalaxy.eu (i.e. datasets, histories) will be available as long
as they are not deleted by the user. Once marked as deleted the datasets will
be permanently removed within 14 days. If the user "purges" the dataset in the
Galaxy, it will be removed immediately, permanently.

An <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9w2MOS6KOlu9XdhRSDqWnCDkzoVBqHJ3zH_My4p8D8ZgkIQ/viewform" target="_blank">extended quota can be requested</a>
for a limited time period in special cases.

<h4>Unregistered Users</h4>

Processed data will only be accessible during one browser session, using a
cookie to identify your data. This cookie is not used for any other purposes
(e.g. tracking or analytics).

If UseGalaxy.eu service is not accessed for 90 days, those datasets will be
permanently deleted.

<h4>FTP Data</h4>

Any user data uploaded to our <a href="https://galaxyproject.eu/ftp/">FTP server</a> should be imported into Galaxy as soon
as possible. Data left in FTP folders for more than 3 months, will be deleted.

<h4>GDPR Compliance</h4>

The Galaxy service complies with the EU General Data Protection Regulation
(GDPR). You can read more about this on our
<a href="https://usegalaxy.eu/terms/">Terms and Conditions</a>. 
 -->

<div>
    <iframe style="border: 0px" width="100%" height="150px" src="https://stats.galaxyproject.eu/d-solo/000000034/jobs-dashboard?orgId=1&refresh=1m&panelId=1" ></iframe>
</div>
<div>
<!--<iframe style="border: 0px" width="33%" height="100px" src="https://stats.galaxyproject.eu/d-solo/000000034/jobs-dashboard?orgId=1&refresh=1m&panelId=3" ></iframe>-->
</div>

<div class="row">

    <section class="section-content">
        <div class="col-md-12">
        </div>
    </section>
</div>

</div>

        </div>
      </div>
    </div>
    







<footer class="navbar-default">
    <div class="container">
        <div class="row">
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <p>UseGalaxy.eu is maintained largely by the <a href="/freiburg/">Freiburg Galaxy Team</a> but also collectively by groups and individuals from across Europe. All of the member sites in this repository contribute to the European Galaxy Project.
For <strong>acknowledgement</strong>, please refer to the <a href="/about">About</a> section.
All content on this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a> unless otherwise specified.</p>

            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu/website/tree/master/index-microgalaxy.md">Edit this page on GitHub</a></li>
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:contact@usegalaxy.eu">contact@usegalaxy.eu</a></li>
                    
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu">usegalaxy-eu</a></li>
                    
                    
                      <li><i class="fa fa-twitter"></i><a href="https://twitter.com/galaxyproject">galaxyproject</a></li>
                    
                      <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li>
                </ul>
            </div>
        </div>
    </div>
</footer>

<script async defer data-domain="galaxyproject.eu" src="https://plausible.galaxyproject.eu/js/plausible.js"></script>
  </body>
</html>
