<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy Europe</title>
  <meta property="og:title" content="" />
  <meta property="og:description" content="" />
  <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" />
  <meta name="description" content="The European Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://galaxyproject.eu/index-microbiome.html">
  <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy Europe" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>

  <body>
    <div id="wrap">
      <div id="main">
        <div class="container" id="maincontainer">
          <div class="home">










<div class="alert alert-danger" style="margin-top: 0.5em">
	<h4>Peace to Ukraine!</h4>
	
	
	<div><p>üá∫üá¶ The global community has created a <a href="https://bit.ly/ua-table" target="_blank">continuously updated list</a> of laboratories that can host Ukrainian scientists at all career levels. If your lab can host a scientist ‚Äì add your name to the list <a href="https://bit.ly/ua-form" target="_blank">here</a>. In addition, Galaxy Project has a number of positions at its EU and US sites. Contact us at Contact us at <a href="mailto:ukraine@galaxyproject.org">ukraine@galaxyproject.org</a>. German organisations are organising accommodation <a href="https://elinor.network/gastfreundschaft-ukraine/" target="_blank">here</a>.</p>

<p>–ù–∞—É–∫–æ–≤–µ —Å—É—Å–ø—ñ–ª—å—Å—Ç–≤–æ —Å—Ç–≤–æ—Ä–∏–ª–æ –ø–æ—Å—Ç—ñ–π–Ω–æ –≤—ñ–¥–Ω–æ–≤–ª—é—î–º–∏–π <a href="https://bit.ly/ua-table" target="_blank">—Å–ø–∏—Å–æ–∫ –ª–∞–±–æ—Ä–∞—Ç–æ—Ä—ñ–π</a>, —è–∫—ñ –º–æ–∂—É—Ç—å –ø—Ä–∏–π–Ω—è—Ç–∏ —É–∫—Ä–∞—ó–Ω—Å—å–∫–∏—Ö –≤—á–µ–Ω–∏—Ö (–≤–∫–ª—é—á–∞—é—á–∏ –∞—Å–ø—ñ—Ä–∞–Ω—Ç—ñ–≤). –î–æ–¥–∞—Ç–∫–æ–≤–æ, Galaxy Project –º–∞—î –≤—ñ–¥–∫—Ä–∏—Ç—ñ –ø–æ–∑–∏—Ü—ñ—ó –Ω–∞ —Å–≤–æƒ´—Ö —î–≤—Ä–æ–ø–µ–π—Å—å–∫–∏—Ö —Ç–∞ –∞–º–µ—Ä–∏–∫–∞–Ω—Å—å–∫–∏—Ö —Å–∞–π—Ç–∞—Ö. –ö–æ–Ω—Ç–∞–∫—Ç—É–π—Ç–µ –Ω–∞—Å –≤–∏–∫–æ—Ä–∏—Å—Ç–æ–≤—É—è <a href="mailto:ukraine@galaxyproject.org">ukraine@galaxyproject.org</a></p>

<p>–ù–∞—É—á–Ω–æ–µ —Å–æ–æ–±—â–µ—Å—Ç–≤–æ —Å–æ–∑–¥–∞–ª–æ –ø–æ—Å—Ç–æ—è–Ω–Ω–æ –æ–±–Ω–æ–≤–ª—è–µ–º—ã–π <a href="https://bit.ly/ua-table" target="_blank">—Å–ø–∏—Å–æ–∫ –ª–∞–±–æ—Ä–∞—Ç–æ—Ä–∏–π</a>, –∫–æ—Ç–æ—Ä—ã–µ –º–æ–≥—É—Ç –ø—Ä–∏–Ω—è—Ç—å —É–∫—Ä–∞–∏–Ω—Å–∫–∏—Ö —É—á–µ–Ω—ã—Ö (–≤–∫–ª—é—á–∞—è –∞—Å–ø–∏—Ä–∞–Ω—Ç–æ–≤). –ö —Ç–æ–º—É –∂–µ, Galaxy Project –∏–º–µ–µ—Ç –æ—Ç–∫—Ä—ã—Ç—ã–µ –ø–æ–∑–∏—Ü–∏–∏ –Ω–∞ —Å–≤–æ–∏—Ö –µ–≤—Ä–æ–ø–µ–π—Å–∫–∏—Ö –∏ –∞–º–µ—Ä–∏–∫–∞–Ω—Å–∫–∏—Ö —Å–∞–π—Ç–∞—Ö.  –ö–æ–Ω—Ç–∞–∫—Ç–∏—Ä—É–π—Ç–µ –Ω–∞—Å –∏—Å–ø–æ–ª—å–∑—É—è <a href="mailto:ukraine@galaxyproject.org">ukraine@galaxyproject.org</a> üá∫üá¶</p>
</div>
	
</div>








<div class="alert alert-info" style="margin-top: 0.5em">
	<h4>Limited computing capacity on next 01.03.2022</h4>
	
	
	<div><p>UPDATE 07.03.2022 17:30 CET, We are slowly returning to normal operation. Not all services are fully functional yet.</p>

<p>UPDATE 05.03.2022 09:00 CET, There are still some issues preventing to release bwCloud for normal activities. bwCloud operating group is working to fix them.</p>

<p>UPDATE 04.03.2022 09:00 CET, bwCloud maintenace is still on going.</p>

<p>UPDATE 03.03.2022 11:00 CET, Usegalaxy.eu is still without compute capacity. We hope to return to normal operation soon.</p>

<p>On March 1st, bwCloud will undergo a scheduled maintenance, and we have to shut down most of our computing resources.
We will keep the web server running, accepting jobs and workflows, but there will be a delay in processing them.
The running jobs, at that date, will be interrupted and rescheduled for a new start (from scratch, no intermediate results can be saved) 
as soon as the cloud is available again.
Please take it into account in your job schedule.</p>

<p>Sorry for the inconvenience.</p>
</div>
	
</div>





  <p><img src="/assets/media/asaim_logo.png" alt="Plant Analysis on Galaxy" class="sc-intro-left" /></p>

<h1 class="no_toc" id="welcome-to-galaxy-for-microbiome">Welcome to Galaxy for Microbiome</h1>
<p><br />
<strong>Galaxy for Microbiome</strong> (<a href="https://asaim.readthedocs.io/en/latest/" target="_blank">ASaiM</a>) is a webserver to process, analyse and visualize microbiome data in general. It is based on the <a href="https://galaxyproject.org" target="_blank">Galaxy framework</a>, which guarantees simple access, easy extension, flexible adaption to personal and security needs, and sophisticated analyses independent of command-line knowledge.</p>

<p><br /></p>
<h1 class="no_toc" id="content">Content</h1>

<ol id="markdown-toc">
  <li><a href="#get-started" id="markdown-toc-get-started">Get started</a></li>
  <li><a href="#training" id="markdown-toc-training">Training</a></li>
  <li><a href="#tools" id="markdown-toc-tools">Tools</a></li>
  <li><a href="#workflows" id="markdown-toc-workflows">Workflows</a>    <ol>
      <li><a href="#taxonomic-and-functional-community-profiling-of-raw-metagenomic-shotgun-data" id="markdown-toc-taxonomic-and-functional-community-profiling-of-raw-metagenomic-shotgun-data">Taxonomic and functional community profiling of raw metagenomic shotgun data</a></li>
      <li><a href="#assembly-of-metagenomic-data" id="markdown-toc-assembly-of-metagenomic-data">Assembly of metagenomic data</a></li>
      <li><a href="#analysis-of-metataxonomic-data" id="markdown-toc-analysis-of-metataxonomic-data">Analysis of metataxonomic data</a></li>
      <li><a href="#asaim-mt-optimized-workflow-for-metatranscriptomics-data-analysis" id="markdown-toc-asaim-mt-optimized-workflow-for-metatranscriptomics-data-analysis">ASaiM-MT: Optimized workflow for metatranscriptomics data analysis</a></li>
      <li><a href="#integrative-meta-omics-analysis---metagenomics-metatranscriptomics-metaproteomics" id="markdown-toc-integrative-meta-omics-analysis---metagenomics-metatranscriptomics-metaproteomics">Integrative meta-omics analysis - Metagenomics, Metatranscriptomics, Metaproteomics</a></li>
    </ol>
  </li>
  <li><a href="#references" id="markdown-toc-references">References</a></li>
</ol>

<h1 id="get-started">Get started</h1>

<p>Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take a <a href="/tours/core.galaxy_ui" target="_blank"><strong>guided tour</strong></a> through Galaxy‚Äôs user interface.</p>

<h1 id="training">Training</h1>

<p>We are working in close collaboration with the <a href="https://training.galaxyproject.org" target="_blank"><strong>Galaxy Training Network (GTN)</strong></a> to develop training materials of data analyses based on Galaxy. If you want to know more about the GTN or how to become part of the Galaxy community, check the videos below!</p>

<iframe width="560" height="315" src="https://www.youtube.com/embed/lDqWxzWNk1k" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
</iframe>

<iframe width="560" height="315" src="https://www.youtube.com/embed/-1MPdxmRs8U" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>

<p><br /><br /></p>

<p><strong>Training material</strong></p>

<p>All relevant materials for <a href="https://training.galaxyproject.org/topics/metagenomics">microbiome data analysis</a> can also be found within the GTN.</p>

<table class="table table-striped">
  <thead>
    <tr>
      <th>Lesson</th>
      <th>Slides</th>
      <th>Hands-on</th>
      <th>Input dataset</th>
      <th>Workflows</th>
      <th>Galaxy History</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Welcome and introduction to Galaxy</td>
      <td><a href="https://training.galaxyproject.org/topics/introduction/tutorials/galaxy-intro-short/slides.html" target="_blank"><i class="fa fa-slideshare" aria-hidden="true"></i></a> / <a href="https://training.galaxyproject.org/videos/watch.html?v=introduction/tutorials/galaxy-intro-short/slides" target="_blank"><i class="fa fa-video-camera" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-short/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://doi.org/10.5281/zenodo.582600" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-short/workflows/" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>An Introduction to Metagenomics</td>
      <td><a href="https://training.galaxyproject.org/topics/metagenomics/slides/introduction.html#1" target="_blank"><i class="fa fa-slideshare" aria-hidden="true"></i></a></td>
      <td>¬†</td>
      <td>¬†</td>
      <td>¬†</td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>Quality Control</td>
      <td><a href="https://training.galaxyproject.org/topics/sequence-analysis/tutorials/quality-control/slides.html" target="_blank"><i class="fa fa-slideshare" aria-hidden="true"></i></a> / <a href="https://youtu.be/BWonTPS4zB8" target="_blank"><i class="fa fa-video-camera" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/topics/sequence-analysis/tutorials/quality-control/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a> / <a href="https://youtu.be/QJRlX2hWDKM" target="_blank"><i class="fa fa-video-camera" aria-hidden="true"></i></a></td>
      <td><a href="https://doi.org/10.5281/zenodo.61771" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/topics/sequence-analysis/tutorials/quality-control/workflows/" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td><a href="/u/gallardoalba/h/quality-control" target="_blank"><i class="fa fa-list-ul" aria-hidden="true"></i></a></td>
    </tr>
    <tr>
      <td>16S Microbial Analysis with mothur</td>
      <td><a href="https://www.youtube.com/watch?v=9OY1mklWuK0" target="_blank"><i class="fa fa-video-camera" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/topics/metagenomics/tutorials/mothur-miseq-sop-short/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i>Short</a> / <a href="https://training.galaxyproject.org/topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i>Extended</a> / <a href="https://youtu.be//mto4Nl-q7Kk" target="_blank"><i class="fa fa-video-camera" aria-hidden="true"></i></a></td>
      <td><a href="https://doi.org/10.5281/zenodo.800651" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/topics/metagenomics/tutorials/mothur-miseq-sop-short/workflows/" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>16S Microbial analysis with Nanopore data</td>
      <td>¬†</td>
      <td><a href="https://training.galaxyproject.org/topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://zenodo.org/record/4274812" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/topics/metagenomics/tutorials/nanopore-16S-metagenomics/workflows/" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>Analyses of metagenomics data - The global picture</td>
      <td>¬†</td>
      <td><a href="https://training.galaxyproject.org/topics/metagenomics/tutorials/general-tutorial/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a></td>
      <td><a href="https://doi.org/10.5281/zenodo.815875" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/topics/metagenomics/tutorials/general-tutorial/workflows/" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/metatranscriptomics/slides.html" target="_blank"><i class="fa fa-slideshare" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/metatranscriptomics-short/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i>Short</a> / <a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/metatranscriptomics/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i>Extended</a></td>
      <td><a href="https://zenodo.org/record/4776250" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/metatranscriptomics-short/workflows/" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
    </tr>
    <tr>
      <td>Metaproteomics introduction</td>
      <td>¬†</td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/proteomics/tutorials/metaproteomics/tutorial.html" target="_blank"><i class="fa fa-laptop" aria-hidden="true"></i></a> / <a href="https://youtu.be//3_yaPp-RCFw" target="_blank"><i class="fa fa-video-camera" aria-hidden="true"></i></a></td>
      <td><a href="https://doi.org/10.5281/zenodo.839701" target="_blank"><i class="fa fa-files-o" aria-hidden="true"></i></a></td>
      <td><a href="https://training.galaxyproject.org/training-material/topics/proteomics/tutorials/metaproteomics/workflows/" target="_blank"><i class="fa fa-share-alt" aria-hidden="true"></i></a></td>
      <td>¬†</td>
    </tr>
  </tbody>
</table>

<h1 id="tools">Tools</h1>

<p>More than <strong>200 tools</strong> are avalaible for microbiome data analysis in this custom Galaxy instance:</p>

<ul>
  <li><strong>General tools</strong>
    <ul>
      <li><strong>Data retrieval</strong>: EBISearch, ENASearch, SRA Tools</li>
      <li><strong>BAM/SAM file manipulation</strong>: SAM tools</li>
      <li><strong>BIOM file manipulation</strong>: BIOM-Format tools</li>
    </ul>
  </li>
  <li><strong>Genomics tools</strong>
    <ul>
      <li><strong>Quality control</strong>: FastQC, PRINSEQ, Cutadapt, fastp, Trimmomatic, MultiQC</li>
      <li><strong>Clustering</strong>: CD-Hit</li>
      <li><strong>Sorting and prediction</strong>: SortMeRNA, FragGeneScan</li>
      <li><strong>Mapping</strong>: BWA, Bowtie</li>
      <li><strong>Similarity search</strong>: NCBI Blast+, Diamond</li>
      <li><strong>Alignment</strong>: HMMER3</li>
    </ul>
  </li>
  <li><strong>Microbiota dedicated tools</strong>
    <ul>
      <li><strong>Microbial</strong>: Scoary, Prokka, Roary</li>
      <li><strong>Metagenomics data manipulation</strong>: VSearch, Nonpareil, DADA2</li>
      <li><strong>Assembly</strong>: MEGAHIT, metaSPAdes, metaQUAST, VALET, Bandage, MaxBin2</li>
      <li><strong>Metataxonomic sequence analysis</strong>: Mothur, QIIME, Vegan</li>
      <li><strong>Taxonomy assignation</strong>: MetaPhlAn, Kraken, CAT/BAT</li>
      <li><strong>Metabolism assignation</strong>: HUMAnN, PICRUST, InterProScan</li>
      <li><strong>Visualization</strong>: Export2graphlan, GraPhlAn, KRONA</li>
      <li><strong>Metaproteomics</strong>: MaxQuant, SearchGUI, PeptideShaker, Unipept</li>
    </ul>
  </li>
</ul>

<h1 id="workflows">Workflows</h1>

<p>To orchestrate tools and help users with their analyses, several <a href="https://asaim.readthedocs.io/en/latest/workflows.html" target="_blank">workflows</a> are available. They formally orchestrate tools in a defined order and with defined parameters, but they are customizable (tools, order, parameters).</p>

<p>The workflows are available in the <a href="/workflows/list_published">Shared Workflows</a>, with the label ‚Äú<strong><em>asaim</em></strong>‚Äù.</p>

<h2 id="taxonomic-and-functional-community-profiling-of-raw-metagenomic-shotgun-data">Taxonomic and functional community profiling of raw metagenomic shotgun data</h2>

<p>The workflow quickly produces, from raw metagenomic shotgun data, accurate and precise taxonomic assignations, wide extended functional results and taxonomically related metabolism information</p>

<p><img src="/assets/media/2018-01-17-asaim_main_workflow.png" alt="ASaiM main workflow" width="75%" /></p>

<p>This workflow consists of</p>

<ol>
  <li>Processing with quality control/trimming (<strong>FastQC</strong> and <strong>Trim Galore!</strong>) and dereplication (<strong>VSearch</strong>)</li>
  <li>Taxonomic analyses with assignation (<strong>MetaPhlAn2</strong>) and visualization (<strong>KRONA</strong>, <strong>GraPhlAn</strong>)</li>
  <li>Functional analyses with metabolic assignation and pathway reconstruction (<strong>HUMAnN2</strong>)</li>
  <li>Functional and taxonomic combination with developed tools combining HUMAnN2 and MetaPhlAn2 outputs</li>
</ol>

<p>It is available with 4 versions, given the input</p>

<ol>
  <li>Simple files: <a href="/u/berenice/w/asaim-shotgun-workflow">Single-end</a> or <a href="/u/berenice/w/asaim---shotgun-workflow-for-paired-end-data">paired-end</a></li>
  <li>Collection input files: <a href="/u/berenice/w/asaim-shotgun-workflow-se-collection">Single-end</a> or <a href="/u/berenice/w/asaim---shotgun-workflow-for-paired-end-data-collection">paired-end</a></li>
</ol>

<h2 id="assembly-of-metagenomic-data">Assembly of metagenomic data</h2>

<p>To reconstruct genomes or to get longer sequences for further analysis, microbiota data needs to be assembled, using the recently developed metagenomics assemblers.</p>

<p>To help in this task, two workflows have been developed using two different well-performing assemblers:</p>

<ul>
  <li>
    <p><a href="/u/berenice/w/asaim-metagenomic-assembly-with-megahit">MEGAHIT</a></p>

    <p>It is currently the most efficent computationally assembler: it has the lowest memory and time consumption <a class="citation" href="#van2017assembling">(van der Walt <i>et al.</i>, 2017; Awad <i>et al.</i>, 2017; Sczyrba <i>et al.</i>, 2017)</a>. It produced some of the best assemblies (irrespective of sequencing coverage) with the fewest structural errors <a class="citation" href="#olson2017metagenomic">(Olson <i>et al.</i>, 2017)</a> and outperforms in recovering the genomes of closely related strains <a class="citation" href="#awad2017evaluating">(Awad <i>et al.</i>, 2017)</a>, but has a bias towards relatively low coverage genomes leading to a suboptimal assembly of high abundant community member genomes in very large datasets <a class="citation" href="#vollmers2017comparing">(Vollmers <i>et al.</i>, 2017)</a></p>
  </li>
  <li>
    <p><a href="/u/berenice/w/asaim-metagenomic-assembly-with-metaspades">MetaSPAdes</a></p>

    <p>It is particularly optimal for high-coverage metagenomes <a class="citation" href="#van2017assembling">(van der Walt <i>et al.</i>, 2017)</a> with the best contig metrics <a class="citation" href="#greenwald2017utilization">(Greenwald <i>et al.</i>, 2017)</a> and produces few under-collapsed/over-collapsed repeats <a class="citation" href="#olson2017metagenomic">(Olson <i>et al.</i>, 2017)</a></p>
  </li>
</ul>

<p>Both workflows consists of</p>

<ol>
  <li>Processing with quality control/trimming (<strong>FastQC</strong> and <strong>Trim Galore!</strong>)</li>
  <li>Assembly with either <strong>MEGAHIT</strong> or <strong>MetaSPAdes</strong></li>
  <li>Estimation of the assembly quality statistics with <strong>MetaQUAST</strong></li>
  <li>Identification of potential assembly error signature with <strong>VALET</strong></li>
  <li>Determination of percentage of unmapped reads with <strong>Bowtie2</strong> combined with <strong>MultiQC</strong> to aggregate the results.</li>
</ol>

<p><img src="/assets/media/microbiome_assembly_wf.png" alt="ASaiM assembly workflows" width="75%" /></p>

<h2 id="analysis-of-metataxonomic-data">Analysis of metataxonomic data</h2>

<p>To analyze amplicon data, the <strong>Mothur</strong> and <strong>QIIME</strong> tool suites are available there. We implemented the workflows described in tutorials of Mothur and QIIME websites, as example of amplicon data analyses as well as support for the training material. These workflows, as any workflows available there, can be adapted for a specific analysis or used as subworkflows by the users.</p>

<h2 id="asaim-mt-optimized-workflow-for-metatranscriptomics-data-analysis">ASaiM-MT: Optimized workflow for metatranscriptomics data analysis</h2>

<p>While the shotgun workflow is suitable for both metagenomics and metatranscriptomics datasets, we also offer an enhanced workflow aimed specifically at metatranscriptomics data.</p>

<p><img src="/assets/media/asaim/asaim-mt_workflow.png" alt="ASaiM-MT workflow" width="50%" /></p>

<p>The workflow is divided into 4 parts:</p>

<ol>
  <li><strong>Preprocessing</strong> - Process raw metatranscriptomics data to perform further analysis.</li>
  <li><strong>Taxonomy Quantitation</strong> - Assignment of taxonomy along with abundance values and visualization.</li>
  <li><strong>Functional Quantitation</strong> - metabolic assignment of identified functions and gene and pathway abundance annotation.</li>
  <li><strong>Taxonomy-Function Quantitation</strong> - combine taxonomy and functional quantitation values into relative abundance values at different levels such as e.g. the abundance of a pathway between phyla.</li>
</ol>

<h2 id="integrative-meta-omics-analysis---metagenomics-metatranscriptomics-metaproteomics">Integrative meta-omics analysis - Metagenomics, Metatranscriptomics, Metaproteomics</h2>

<p>The combination of metagenomics, -transcriptomics and -proteomics can provide a detailed understanding of which organisms occupy specific metabolic niches, how they interact, and how they utilize environmental nutrients. Commonly used omics tools spanning metagenomics, -transcriptomics and -proteomics has been adapted into an integrated meta-omics analysis pipeline:</p>

<ul>
  <li>
    <p><strong>Metagenomics</strong></p>

    <p><img src="/assets/media/magnus/metagenomics.png" alt="Magnus metagenomics workflow" /></p>
  </li>
  <li>
    <p><strong>Metatranscriptomics</strong></p>

    <p><img src="/assets/media/magnus/metatranscriptomics.png" alt="Magnus metatranscriptomics workflow" width="75%" /></p>
  </li>
  <li>
    <p><strong>Metaproteomics</strong></p>

    <p><img src="/assets/media/magnus/metaproteomics.png" alt="Magnus metaproteomics workflow" width="50%" /></p>
  </li>
  <li>
    <p>Integration of omics data using R</p>
  </li>
</ul>

<p>This pipeline has been applied to <a href="/posts/2020/04/14/integrative-meta-omics/">deconvolute a highly efficient cellulose-degrading minimal consortium isolated and enriched from a biogas reactor in Fredrikstad, Norway</a></p>

<h1 id="references">References</h1>

<ol class="bibliography"><li><span id="index-metagenomics-awad2017evaluating">Awad,S. <i>et al.</i> (2017) Evaluating Metagenome Assembly on a Simple Defined Community with Many Strain Variants. <i>bioRxiv</i>, 155358.</span></li>
<li><span id="index-metagenomics-greenwald2017utilization">Greenwald,W.W. <i>et al.</i> (2017) Utilization of defined microbial communities enables effective evaluation of meta-genomic assemblies. <i>BMC genomics</i>, <b>18</b>, 296.</span></li>
<li><span id="index-metagenomics-olson2017metagenomic">Olson,N.D. <i>et al.</i> (2017) Metagenomic assembly through the lens of validation: recent advances in assessing and improving the quality of genomes assembled from metagenomes. <i>Briefings in Bioinformatics</i>, bbx098.</span></li>
<li><span id="index-metagenomics-sczyrba2017critical">Sczyrba,A. <i>et al.</i> (2017) Critical Assessment of Metagenome Interpretation- a benchmark of computational metagenomics software. <i>Biorxiv</i>, 099127.</span></li>
<li><span id="index-metagenomics-van2017assembling">Walt,A.J. van der <i>et al.</i> (2017) Assembling metagenomes, one community at a time. <i>bioRxiv</i>, 120154.</span></li>
<li><span id="index-metagenomics-vollmers2017comparing">Vollmers,J. <i>et al.</i> (2017) Comparing and Evaluating Metagenome Assembly Tools from a Microbiologist‚Äôs Perspective-Not Only Size Matters! <i>PloS one</i>, <b>12</b>, e0169662.</span></li></ol>


<h2>Our Data Policy</h2>


  <table class="table table-striped">
    <thead>
      <tr>
        <th>Registered Users</th><th>Unregistered Users</th><th>FTP Data</th><th>GDPR Compliance</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>User data on UseGalaxy.eu (i.e. datasets, histories) will be available as long
            as they are not deleted by the user. Once marked as deleted the datasets will
            be permanently removed within 14 days. If the user "purges" the dataset in the
            Galaxy, it will be removed immediately, permanently.

            An <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9w2MOS6KOlu9XdhRSDqWnCDkzoVBqHJ3zH_My4p8D8ZgkIQ/viewform" target="_blank">extended quota can be requested</a>
for a limited time period in special cases.
        </td>
        <td>Processed data will only be accessible during one browser session, using a
            cookie to identify your data. This cookie is not used for any other purposes
            (e.g. tracking or analytics).
            
            If UseGalaxy.eu service is not accessed for 90 days, those datasets will be
            permanently deleted.
        </td>
        <td>Any user data uploaded to our <a href="https://galaxyproject.eu/ftp/">FTP server</a> should be imported into Galaxy as soon
            as possible. Data left in FTP folders for more than 3 months, will be deleted.            
        </td>
        <td>The Galaxy service complies with the EU General Data Protection Regulation
            (GDPR). You can read more about this on our
            <a href="https://usegalaxy.eu/terms/">Terms and Conditions</a>.</td>
      </tr>
    </tbody>
  </table>



<!-- <h4>Registered Users</h4>

User data on UseGalaxy.eu (i.e. datasets, histories) will be available as long
as they are not deleted by the user. Once marked as deleted the datasets will
be permanently removed within 14 days. If the user "purges" the dataset in the
Galaxy, it will be removed immediately, permanently.

An <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9w2MOS6KOlu9XdhRSDqWnCDkzoVBqHJ3zH_My4p8D8ZgkIQ/viewform" target="_blank">extended quota can be requested</a>
for a limited time period in special cases.

<h4>Unregistered Users</h4>

Processed data will only be accessible during one browser session, using a
cookie to identify your data. This cookie is not used for any other purposes
(e.g. tracking or analytics).

If UseGalaxy.eu service is not accessed for 90 days, those datasets will be
permanently deleted.

<h4>FTP Data</h4>

Any user data uploaded to our <a href="https://galaxyproject.eu/ftp/">FTP server</a> should be imported into Galaxy as soon
as possible. Data left in FTP folders for more than 3 months, will be deleted.

<h4>GDPR Compliance</h4>

The Galaxy service complies with the EU General Data Protection Regulation
(GDPR). You can read more about this on our
<a href="https://usegalaxy.eu/terms/">Terms and Conditions</a>. 
 -->

<div>
    <iframe style="border: 0px" width="100%" height="150px" src="https://stats.galaxyproject.eu/d-solo/000000034/jobs-dashboard?orgId=1&refresh=1m&panelId=1" ></iframe>
</div>
<div>
<!--<iframe style="border: 0px" width="33%" height="100px" src="https://stats.galaxyproject.eu/d-solo/000000034/jobs-dashboard?orgId=1&refresh=1m&panelId=3" ></iframe>-->
</div>

<div class="row">

    <section class="section-content">
        <div class="col-md-12">
        </div>
    </section>
</div>

</div>

        </div>
      </div>
    </div>
    







<footer class="navbar-default">
    <div class="container">
        <div class="row">
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <p>UseGalaxy.eu is maintained largely by the <a href="/freiburg/">Freiburg Galaxy Team</a> but also collectively by groups and individuals from across Europe. All of the member sites in this repository contribute to the European Galaxy Project.
For <strong>acknowledgement</strong>, please refer to the <a href="/about">About</a> section.
All content on this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a> unless otherwise specified.</p>

            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu/website/tree/master/index-microbiome.md">Edit this page on GitHub</a></li>
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:contact@usegalaxy.eu">contact@usegalaxy.eu</a></li>
                    
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu">usegalaxy-eu</a></li>
                    
                    
                      <li><i class="fa fa-twitter"></i><a href="https://twitter.com/galaxyproject">galaxyproject</a></li>
                    
                      <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li>
                </ul>
            </div>
        </div>
    </div>
</footer>

<script async defer data-domain="galaxyproject.eu" src="https://plausible.galaxyproject.eu/js/plausible.js"></script>
  </body>
</html>
