<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy Europe | UseGalaxy.eu Tool Updates for 2020-05-15</title>
  <meta property="og:title" content="UseGalaxy.eu Tool Updates for 2020-05-15" />
  <meta property="og:description" content="On 2020-05-15, the tools on UseGalaxy.eu were updated by ..." />
  <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" />
  <meta name="description" content="The European Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://galaxyproject.eu/posts/2020/05/15/tool-update/plain.html">
  <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy Europe" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>

  <body>
    <div id="wrap">
      <div id="main">
        <div class="container" id="maincontainer">
          




<section id="news-page">
  <div class="row news-header-row">
    
    <a class="col-xs-1 text-center" href="/posts/2020/05/14/galactic-news/plain.html">
      «
    </a>
    
    <div class="col-xs-10 ">
      <div class="text-center">
        <h2 class="title">UseGalaxy.eu Tool Updates for 2020-05-15</h2>
      </div>
      <hr />
    </div>
    
    <a class="col-xs-1 text-center" href="/posts/2020/05/24/tool-update/plain.html">
      »
    </a>
    
  </div>
  <div class="row post-metadata">
    <div class="col-md-8 col-md-offset-2">
      <div class="row">
        <div class="col-md-6">
            

        </div>
        <div class="col-md-6 text-right">
          <div>
            <span class="post-date">
              May 15, 2020
            </span>

            

            <br />
            <span class="post-tags">
                <span class="label label-success">Freiburg</span>
              
                <span class="label label-info">tools</span>
              
              
            </span>

            
          </div>
        </div>
      </div>
    </div>
  </div>

  <div class="row">
    <div class="col-md-8 col-md-offset-2">
      <div class="news-content">
        <p>On 2020-05-15, the tools on UseGalaxy.eu were updated by our automated tool update and installation process in <a href="https://build.galaxyproject.eu/job/usegalaxy-eu/job/install-tools/#211/">Jenkins Build #211</a></p>

<h2 id="hca_sc_scmap_tools">hca_sc_scmap_tools</h2>

<ul>
  <li>scmap_get_std_output was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scmap_get_std_output/39370cc49ae7">39370cc49ae7</a></li>
  <li>scmap_scmap_cell was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scmap_scmap_cell/b78cdc83a479">b78cdc83a479</a></li>
</ul>

<h2 id="hca_sc_monocle3_tools">hca_sc_monocle3_tools</h2>

<ul>
  <li>monocle3_create was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_create/5a4b2fcd04d1">5a4b2fcd04d1</a></li>
  <li>monocle3_diffexp was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_diffexp/d333006ef72a">d333006ef72a</a></li>
  <li>monocle3_learngraph was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_learngraph/17e74fba97fd">17e74fba97fd</a></li>
  <li>monocle3_learngraph was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_learngraph/8ec85828b6e6">8ec85828b6e6</a></li>
  <li>monocle3_ordercells was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_ordercells/a14bd9053e2b">a14bd9053e2b</a></li>
  <li>monocle3_partition was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_partition/92ca41fbc123">92ca41fbc123</a></li>
  <li>monocle3_partition was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_partition/b253a59cbb98">b253a59cbb98</a></li>
  <li>monocle3_plotcells was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_plotcells/016c39d243fb">016c39d243fb</a></li>
  <li>monocle3_preprocess was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_preprocess/181ef11cba9d">181ef11cba9d</a></li>
  <li>monocle3_reducedim was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_reducedim/89aea3f27ed2">89aea3f27ed2</a></li>
  <li>monocle3_topmarkers was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_topmarkers/bc2df96ef963">bc2df96ef963</a></li>
</ul>

<h2 id="metagenomic-analysis">Metagenomic Analysis</h2>

<ul>
  <li>drep_dereplicate was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/drep_dereplicate/8dfcdbeaeed8">8dfcdbeaeed8</a></li>
</ul>

<h2 id="hca_sc_label_analysis_tools">hca_sc_label_analysis_tools</h2>

<ul>
  <li>ct_build_cell_ontology_dict was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/ct_build_cell_ontology_dict/982a151b2960">982a151b2960</a></li>
  <li>ct_combine_tool_outputs was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/ct_combine_tool_outputs/44b6219cba2d">44b6219cba2d</a></li>
  <li>ct_get_consensus_outputs was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/ct_get_consensus_outputs/559fbff1f573">559fbff1f573</a></li>
  <li>ct_get_empirical_dist was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/ct_get_empirical_dist/ce70358b0fe3">ce70358b0fe3</a></li>
  <li>ct_get_tool_perf_table was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/ct_get_tool_perf_table/71c809e81f66">71c809e81f66</a></li>
  <li>ct_get_tool_pvals was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/ct_get_tool_pvals/39f348e76f41">39f348e76f41</a></li>
</ul>

<h2 id="hca_sc_garnett_tools">hca_sc_garnett_tools</h2>

<ul>
  <li>garnett_check_markers was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/garnett_check_markers/ce67c5c9a56e">ce67c5c9a56e</a></li>
  <li>garnett_classify_cells was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/garnett_classify_cells/715b020ecfb6">715b020ecfb6</a></li>
  <li>garnett_get_feature_genes was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/garnett_get_feature_genes/3176cbad2800">3176cbad2800</a></li>
  <li>garnett_get_std_output was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/garnett_get_std_output/5f05e2901299">5f05e2901299</a></li>
  <li>garnett_train_classifier was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/garnett_train_classifier/9be21507358d">9be21507358d</a></li>
  <li>garnett_transform_markers was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/garnett_transform_markers/1dc49d16e96c">1dc49d16e96c</a></li>
  <li>update_marker_file was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/update_marker_file/3a1cfbb8aa56">3a1cfbb8aa56</a></li>
</ul>

<h2 id="proteomics">Proteomics</h2>

<ul>
  <li>cardinal_classification was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/cardinal_classification/277dc652246e">277dc652246e</a></li>
  <li>cardinal_classification was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/cardinal_classification/47fc5b518ffc">47fc5b518ffc</a></li>
  <li>cardinal_combine was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/cardinal_combine/525f201b86c1">525f201b86c1</a></li>
  <li>cardinal_data_exporter was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/cardinal_data_exporter/653e66bd9eed">653e66bd9eed</a></li>
  <li>cardinal_filtering was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/cardinal_filtering/a8157342d9a3">a8157342d9a3</a></li>
  <li>cardinal_mz_images was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/cardinal_mz_images/a69865da6f80">a69865da6f80</a></li>
  <li>cardinal_preprocessing was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/cardinal_preprocessing/87bb011a4ee8">87bb011a4ee8</a></li>
  <li>cardinal_quality_report was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/cardinal_quality_report/0d4d4f16d455">0d4d4f16d455</a></li>
  <li>cardinal_segmentations was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/cardinal_segmentations/b591450b3d1c">b591450b3d1c</a></li>
  <li>cardinal_spectra_plots was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/cardinal_spectra_plots/ebb7de263d49">ebb7de263d49</a></li>
</ul>

<h2 id="hca_sc_scpred_tools">hca_sc_scpred_tools</h2>

<ul>
  <li>scpred_eigen_decompose was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scpred_eigen_decompose/3867289e6c1a">3867289e6c1a</a></li>
  <li>scpred_get_feature_space was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scpred_get_feature_space/4a77d2091aec">4a77d2091aec</a></li>
  <li>scpred_get_std_output was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scpred_get_std_output/43ce84bd1d9a">43ce84bd1d9a</a></li>
  <li>scpred_predict_labels was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scpred_predict_labels/1bc003e80986">1bc003e80986</a></li>
  <li>scpred_train_model was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scpred_train_model/ba53e46a4275">ba53e46a4275</a></li>
  <li>scpred_traint_test_split was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scpred_traint_test_split/a6d66c9953ed">a6d66c9953ed</a></li>
</ul>

<h2 id="rna-analysis">RNA Analysis</h2>

<ul>
  <li>tetoolkit_tetranscripts was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/tetoolkit_tetranscripts/2dfbcb88d16a">2dfbcb88d16a</a></li>
</ul>

<h2 id="hca_sc_scmap_tools-1">hca_sc_scmap_tools</h2>

<ul>
  <li>scmap_index_cell was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scmap_index_cell/11b4b9b85b9c">11b4b9b85b9c</a></li>
  <li>scmap_index_cluster was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scmap_index_cluster/38c6a98c7c60">38c6a98c7c60</a></li>
  <li>scmap_scmap_cell was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scmap_scmap_cell/5e6d54e2ab89">5e6d54e2ab89</a></li>
  <li>scmap_scmap_cluster was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scmap_scmap_cluster/49025c0cd5ca">49025c0cd5ca</a></li>
  <li>scmap_select_features was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scmap_select_features/139321321fdd">139321321fdd</a></li>
</ul>

<h2 id="phylogenetics">Phylogenetics</h2>

<ul>
  <li>hyphy_absrel was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/hyphy_absrel/c78f3d7dd1db">c78f3d7dd1db</a></li>
  <li>hyphy_bgm was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/hyphy_bgm/7fd2d1451728">7fd2d1451728</a></li>
  <li>hyphy_busted was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/hyphy_busted/686a8b41d09c">686a8b41d09c</a></li>
  <li>hyphy_fade was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/hyphy_fade/bf27b3df1a4f">bf27b3df1a4f</a></li>
  <li>hyphy_fel was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/hyphy_fel/ae95a6b22562">ae95a6b22562</a></li>
  <li>hyphy_fubar was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/hyphy_fubar/ff1e9dd7ad13">ff1e9dd7ad13</a></li>
  <li>hyphy_gard was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/hyphy_gard/7a2092844346">7a2092844346</a></li>
  <li>hyphy_meme was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/hyphy_meme/b69685ed772e">b69685ed772e</a></li>
  <li>hyphy_prime was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/hyphy_prime/e06961ac014b">e06961ac014b</a></li>
  <li>hyphy_relax was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/hyphy_relax/2ccc579efdde">2ccc579efdde</a></li>
  <li>hyphy_slac was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/hyphy_slac/4ddef72f7f21">4ddef72f7f21</a></li>
  <li>hyphy_sm19 was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/hyphy_sm19/c102811766ef">c102811766ef</a></li>
  <li>sarscov2formatter was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/sarscov2formatter/1c664ff29354">1c664ff29354</a></li>
  <li>sarscov2summary was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/sarscov2summary/4ad8b610ea41">4ad8b610ea41</a></li>
</ul>

<h2 id="convert-formats">Convert Formats</h2>

<ul>
  <li>thermo_raw_file_converter was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/thermo_raw_file_converter/63769c4217a7">63769c4217a7</a></li>
</ul>

<h2 id="proteomics-1">Proteomics</h2>

<ul>
  <li>maxquant was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/maxquant/2133b0be850a">2133b0be850a</a></li>
  <li>proteomics_moff was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/proteomics_moff/a96af68dafb2">a96af68dafb2</a></li>
</ul>

<h2 id="metagenomic-analysis-1">Metagenomic Analysis</h2>

<ul>
  <li>drep_compare was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/drep_compare/ef1c257adcbd">ef1c257adcbd</a></li>
  <li>drep_dereplicate was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/drep_dereplicate/8dfcdbeaeed8">8dfcdbeaeed8</a></li>
</ul>

<h2 id="nanopore">Nanopore</h2>

<ul>
  <li>nanocompore_sampcomp was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/nanocompore_sampcomp/557cf45ff2c8">557cf45ff2c8</a></li>
  <li>nanopolishcomp_eventaligncollapse was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/nanopolishcomp_eventaligncollapse/ad011fc670d6">ad011fc670d6</a></li>
  <li>nanopolishcomp_freqmethcalculate was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/nanopolishcomp_freqmethcalculate/80b6e23cb70d">80b6e23cb70d</a></li>
</ul>

<h2 id="statistics">Statistics</h2>

<ul>
  <li>create_tool_recommendation_model was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/create_tool_recommendation_model/5b3c08710e47">5b3c08710e47</a></li>
</ul>

<h2 id="imaging">Imaging</h2>

<ul>
  <li>cp_cellprofiler was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_cellprofiler/0b8f8406ee16">0b8f8406ee16</a></li>
  <li>cp_cellprofiler was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_cellprofiler/a98608f3abc5">a98608f3abc5</a></li>
  <li>cp_common was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_common/0b7db679fff8">0b7db679fff8</a></li>
  <li>cp_common was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_common/27396311a314">27396311a314</a></li>
  <li>cp_convert_objects_to_image was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_convert_objects_to_image/2690da374874">2690da374874</a></li>
  <li>cp_display_data_on_image was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_display_data_on_image/4e12ef887a2a">4e12ef887a2a</a></li>
  <li>cp_enhance_or_suppress_features was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_enhance_or_suppress_features/29188f9c4ca8">29188f9c4ca8</a></li>
  <li>cp_enhance_or_suppress_features was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_enhance_or_suppress_features/3f9747bda8c4">3f9747bda8c4</a></li>
  <li>cp_export_to_spreadsheet was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_export_to_spreadsheet/10257d6d545b">10257d6d545b</a></li>
  <li>cp_export_to_spreadsheet was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_export_to_spreadsheet/72a7f23f3a79">72a7f23f3a79</a></li>
  <li>cp_gray_to_color was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_gray_to_color/bbf1ef17c01b">bbf1ef17c01b</a></li>
  <li>cp_identify_primary_objects was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_identify_primary_objects/52c01fb29b2a">52c01fb29b2a</a></li>
  <li>cp_image_math was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_image_math/6d3851ac885f">6d3851ac885f</a></li>
  <li>cp_mask_image was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_mask_image/54d7d3450800">54d7d3450800</a></li>
  <li>cp_mask_image was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_mask_image/ca1421f7e3ed">ca1421f7e3ed</a></li>
  <li>cp_measure_granularity was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_measure_granularity/1062c9e44147">1062c9e44147</a></li>
  <li>cp_measure_granularity was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_measure_granularity/382413f8ec1d">382413f8ec1d</a></li>
  <li>cp_measure_image_area_occupied was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_measure_image_area_occupied/7a08edbc790e">7a08edbc790e</a></li>
  <li>cp_measure_image_intensity was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_measure_image_intensity/2681008d726f">2681008d726f</a></li>
  <li>cp_measure_image_quality was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_measure_image_quality/08a2cf887a44">08a2cf887a44</a></li>
  <li>cp_measure_object_intensity was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_measure_object_intensity/1373b765c8f9">1373b765c8f9</a></li>
  <li>cp_measure_object_intensity was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_measure_object_intensity/b867e9317a06">b867e9317a06</a></li>
  <li>cp_measure_object_size_shape was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_measure_object_size_shape/5c9549983af8">5c9549983af8</a></li>
  <li>cp_measure_texture was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_measure_texture/0f751bb492de">0f751bb492de</a></li>
  <li>cp_measure_texture was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_measure_texture/658a2a451aec">658a2a451aec</a></li>
  <li>cp_relate_objects was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_relate_objects/3a2cfcd7c38a">3a2cfcd7c38a</a></li>
  <li>cp_relate_objects was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_relate_objects/f3f45c42bc36">f3f45c42bc36</a></li>
  <li>cp_save_images was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_save_images/6e0f244acd32">6e0f244acd32</a></li>
  <li>cp_save_images was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/cp_save_images/a45d360ae9d9">a45d360ae9d9</a></li>
  <li>projective_transformation_points was updated to <a href="https://toolshed.g2.bx.psu.edu/view/imgteam/projective_transformation_points/a84822a0060c">a84822a0060c</a></li>
  <li>idr_download_by_ids was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/idr_download_by_ids/11036f6197d6">11036f6197d6</a></li>
</ul>

<h2 id="hca_sc_monocle3_tools-1">hca_sc_monocle3_tools</h2>

<ul>
  <li>monocle3_diffexp was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_diffexp/c5665e286bcf">c5665e286bcf</a></li>
  <li>monocle3_learngraph was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_learngraph/6b6b5a6d4021">6b6b5a6d4021</a></li>
  <li>monocle3_ordercells was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_ordercells/762c57e35238">762c57e35238</a></li>
  <li>monocle3_partition was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_partition/60cedaee808d">60cedaee808d</a></li>
  <li>monocle3_plotcells was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/monocle3_plotcells/ef2dc754709c">ef2dc754709c</a></li>
</ul>

<h2 id="variant-calling">Variant Calling</h2>

<ul>
  <li>gatk4_mutect2 was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/gatk4_mutect2/287c4af133e6">287c4af133e6</a></li>
</ul>

<h2 id="rna-analysis-1">RNA Analysis</h2>

<ul>
  <li>chira_collapse was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/chira_collapse/c319651a733a">c319651a733a</a></li>
  <li>chira_extract was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/chira_extract/89ab3b2973d8">89ab3b2973d8</a></li>
  <li>chira_map was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/chira_map/1d3be7700f70">1d3be7700f70</a></li>
  <li>chira_merge was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/chira_merge/35f433d0bd7f">35f433d0bd7f</a></li>
  <li>chira_quantify was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/chira_quantify/a1694947910f">a1694947910f</a></li>
  <li>seurat was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/seurat/82fcdf530f87">82fcdf530f87</a></li>
  <li>slamdunk was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/slamdunk/24657d5edede">24657d5edede</a></li>
</ul>

<h2 id="chemicaltoolbox">ChemicalToolBox</h2>

<ul>
  <li>ctb_frankenstein_ligand was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/ctb_frankenstein_ligand/7a4306d69801">7a4306d69801</a></li>
  <li>rdock_rbcavity was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/rdock_rbcavity/06f376dc117a">06f376dc117a</a></li>
  <li>rdock_rbcavity was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/rdock_rbcavity/82193638d0c5">82193638d0c5</a></li>
  <li>rdock_rbdock was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/rdock_rbdock/309fd04bcfd2">309fd04bcfd2</a></li>
  <li>rdock_sort_filter was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/rdock_sort_filter/a6ec6c55267e">a6ec6c55267e</a></li>
  <li>rdock_sort_filter was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/rdock_sort_filter/c88da0252e85">c88da0252e85</a></li>
  <li>gmx_editconf was updated to <a href="https://toolshed.g2.bx.psu.edu/view/chemteam/gmx_editconf/259a1b8a1a39">259a1b8a1a39</a></li>
  <li>gmx_em was updated to <a href="https://toolshed.g2.bx.psu.edu/view/chemteam/gmx_em/dc98e5c8165a">dc98e5c8165a</a></li>
  <li>gmx_energy was updated to <a href="https://toolshed.g2.bx.psu.edu/view/chemteam/gmx_energy/345c2a16b95f">345c2a16b95f</a></li>
  <li>gmx_makendx was updated to <a href="https://toolshed.g2.bx.psu.edu/view/chemteam/gmx_makendx/c119d1b9941b">c119d1b9941b</a></li>
  <li>gmx_merge_topology_files was updated to <a href="https://toolshed.g2.bx.psu.edu/view/chemteam/gmx_merge_topology_files/d1d0efda35c0">d1d0efda35c0</a></li>
  <li>gmx_restraints was updated to <a href="https://toolshed.g2.bx.psu.edu/view/chemteam/gmx_restraints/a550cfa43262">a550cfa43262</a></li>
  <li>gmx_setup was updated to <a href="https://toolshed.g2.bx.psu.edu/view/chemteam/gmx_setup/8b2553f4f114">8b2553f4f114</a></li>
  <li>gmx_sim was updated to <a href="https://toolshed.g2.bx.psu.edu/view/chemteam/gmx_sim/4b4232c98b6e">4b4232c98b6e</a></li>
  <li>gmx_solvate was updated to <a href="https://toolshed.g2.bx.psu.edu/view/chemteam/gmx_solvate/dc2400b6361d">dc2400b6361d</a></li>
  <li>gmx_trj was updated to <a href="https://toolshed.g2.bx.psu.edu/view/chemteam/gmx_trj/6ac5f450ad31">6ac5f450ad31</a></li>
</ul>


      </div>
    </div>
  </div>


<div id="supported">
    
  <h3>Supported by</h3>
    
  <div class="row eu-image-box">
      
      <div>
      
          
            <a href="https://www.denbi.de" target="_blank">
              <img alt="de.NBI logo" src="/assets/media/deNBI_Logo_rgb_small.png" class="img-responsive eu-image"/>
            </a>
          
      </div>
      
      <div>
      
          
            <a href="https://www.elixir-europe.org" target="_blank">
              <img alt="ELIXIR logo" src="/assets/media/elixir_logo.png" class="img-responsive eu-image" />
            </a>
          
      </div>
      
  </div>
</div>


</section>

        </div>
      </div>
    </div>
    







<footer class="navbar-default">
    <div class="container">
        <div class="row">
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <p>The <a href="/freiburg/people">Freiburg Galaxy Team</a> is headed by Björn Grüning
and is part of the <a href="http://www.bioinf.uni-freiburg.de">Bioinformatics Group
Freiburg</a>, headed by Prof. Dr. Rolf
Backofen.</p>

<p>The team is part of <a href="https://www.denbi.de">de.NBI</a> and
<a href="https://www.elixir-europe.org">ELIXIR</a> and largely funded by
<a href="http://dfg.de/">DFG</a> and <a href="https://www.bmbf.de">BMBF</a>. All content on
this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a>,
unless otherwise specified.</p>

            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu/website/tree/master/_posts_plain/2020-05-15-tool-update.md">Edit this page on GitHub</a></li>
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:galaxy@informatik.uni-freiburg.de">galaxy@informatik.uni-freiburg.de</a></li>
                    
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu">usegalaxy-eu</a></li>
                    
                    
                      <li><i class="fa fa-twitter"></i><a href="https://twitter.com/galaxyproject">galaxyproject</a></li>
                    
                      <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li>
                </ul>
            </div>
        </div>
    </div>
</footer>

  </body>
</html>
