<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy Europe</title>
  <meta property="og:title" content="" />
  <meta property="og:description" content="" />
  <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" />
  <meta name="description" content="The European Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://galaxyproject.eu/freiburg/publications.html">
  <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy Europe" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>


    
    
    

  <body class="location-freiburg">
    <div id="wrap">
      <div id="main">
        <div class="navbar">

  <div class="container">
    
    

    
    
    

    

    <div class="navbar-header">
      <div class="navbar-brand">
        
        <img alt="galaxy eu logo" src="/assets/media/galaxy-eu-logo.inv.svg" style="height: 20px"/>
        
      </div>

      <a class="navbar-brand" href="/" style="padding-left: 6px">
        Galaxy Europe
      </a>
      
        <span class="navbar-brand" style="color: white">
        /
        </span>
        <a class="navbar-brand" href="https://galaxyproject.eu/freiburg">
          Freiburg
        </a>
      
    </div>

      <div class="navbar-collapse collapse">
        <ul class="nav navbar-nav navbar-right">
           <li class="dropdown">
             <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="true">Member Sites <span class="caret"></span></a>
             <ul class="dropdown-menu">
              
                <li>
                  <a class="page-link" href="/freiburg/">Freiburg</a>
                </li>
              
                <li>
                  <a class="page-link" href="/erasmusmc/">Erasmus MC</a>
                </li>
              
                <li>
                  <a class="page-link" href="/belgium/">VIB</a>
                </li>
              
                <li>
                  <a class="page-link" href="/pasteur/">Pasteur</a>
                </li>
              
                <li>
                  <a class="page-link" href="/genouest/">GenOuest</a>
                </li>
              
                <li>
                  <a class="page-link" href="/elixir-it/">ELIXIR-IT/Laniakea</a>
                </li>
              
                <li>
                  <a class="page-link" href="/ifb/">ELIXIR-FR/IFB</a>
                </li>
              
             </ul>
           </li>

          
          
            <li>
              <a class="page-link" href="/freiburg/news">News</a>
            </li>
          
            <li>
              <a class="page-link" href="/freiburg/events">Events</a>
            </li>
          
            <li>
              <a class="page-link" href="/freiburg/people">People</a>
            </li>
          
            <li>
              <a class="page-link" href="/freiburg/projects">Projects</a>
            </li>
          
            <li>
              <a class="page-link" href="/freiburg/services">Services</a>
            </li>
          
            <li>
              <a class="page-link" href="/freiburg/jobs">Jobs</a>
            </li>
          
            <li>
              <a class="page-link" href="/freiburg/publications">Publications</a>
            </li>
          
          
        </ul>
      </div>
  </div>
</div>

        <div class="container" id="maincontainer">
          <h1 id="our-publications">Our publications</h1>

<h2 class="bibliography">2025</h2>
<ol class="bibliography"><li><span id="PerezRiverol2025">Perez-Riverol,Y. <i>et al.</i> (2025) Open-Source and FAIR Research Software for Proteomics. <i>Journal of Proteome Research</i>.</span></li>
<li><span id="Fuchs2025">Fuchs,J. <i>et al.</i> (2025) varVAMP: degenerate primer design for tiled full genome sequencing and qPCR. <i>Nature Communications</i>, <b>16</b>.</span></li>
<li><span id="petroll2025">Petroll,R. <i>et al.</i> (2025) Enhanced sensitivity of TAPscan v4 enables comprehensive analysis of streptophyte transcription factor evolution. <i>The Plant Journal</i>, <b>121</b>, e17184.</span></li>
<li><span id="Royaux2025">Royaux,C. <i>et al.</i> (2025) Guidance framework to apply best practices in ecological data analysis: lessons learned from building Galaxy-Ecology. <i>GigaScience</i>, <b>14</b>.</span></li>
<li><span id="Rubin2025">Rubin,A.F. <i>et al.</i> (2025) MaveDB 2024: a curated community database with over seven million variant effects from multiplexed functional assays. <i>Genome Biology</i>, <b>26</b>.</span></li>
<li><span id="goclowski2025galaxy">Goclowski,C.L. <i>et al.</i> (2025) Galaxy as a gateway to bioinformatics: Multi-Interface Galaxy Hands-on Training Suite (MIGHTS) for scRNA-seq. <i>GigaScience</i>, <b>14</b>, giae107.</span></li></ol>
<h2 class="bibliography">2024</h2>
<ol class="bibliography"><li><span id="muller2024cherri">Müller,T. <i>et al.</i> (2024) CheRRI—Accurate classification of the biological relevance of putative RNA–RNA interaction sites. <i>GigaScience</i>, <b>13</b>, giae022.</span></li>
<li><span id="2024">Abueg,L.A.L. <i>et al.</i> (2024) The Galaxy platform for accessible,  reproducible,  and collaborative data analyses: 2024 update. <i>Nucleic Acids Research</i>.</span></li>
<li><span id="Galvis2024">Galvis,J. <i>et al.</i> (2024) DIMet: An open-source tool for Differential analysis of targeted <u>I</u>sotope-labeled Metabolomics data. <i>Bioinformatics</i>.</span></li>
<li><span id="Rahman2024">Rahman,N. <i>et al.</i> (2024) Mobilisation and analyses of publicly available SARS-CoV-2 data for pandemic responses. <i>Microbial Genomics</i>, <b>10</b>.</span></li>
<li><span id="Larivire2024">Larivière,D. <i>et al.</i> (2024) Scalable,  accessible and reproducible reference genome assembly and evaluation in Galaxy. <i>Nature Biotechnology</i>.</span></li>
<li><span id="Finn2024">Finn,R.D. <i>et al.</i> (2024) Establishing the ELIXIR Microbiome Community. <i>F1000Research</i>, <b>13</b>.</span></li></ol>
<h2 class="bibliography">2023</h2>
<ol class="bibliography"><li><span id="peguilhan2023clouds">Péguilhan,R. <i>et al.</i> (2023) Clouds, oases for airborne microbes–Differential metagenomics/metatranscriptomics analyses of cloudy and clear atmospheric situations. <i>bioRxiv</i>, 2023–12.</span></li>
<li><span id="Kumar2023">Kumar,A. <i>et al.</i> (2023) Transformer-based tool recommendation system in Galaxy. <i>BMC Bioinformatics</i>, <b>24</b>.</span></li>
<li><span id="David2023">David,R. <i>et al.</i> (2023) “Be sustainable”: <scp>EOSC‐Life</scp> recommendations for implementation of <scp>FAIR</scp> principles in life science data handling. <i>The EMBO Journal</i>.</span></li>
<li><span id="Williams2023">Williams,J.J. <i>et al.</i> (2023) An international consensus on effective, inclusive, and career-spanning short-format training in the life sciences and beyond. <i>PLOS ONE</i>, <b>18</b>, 1–19.</span></li>
<li><span id="Weil2023">Weil,H.L. <i>et al.</i> (2023) \lessscp\greaterPLANTdataHUB\less/scp\greater: a collaborative platform for continuous \lessscp\greaterFAIR\less/scp\greater data sharing in plant research. <i>The Plant Journal</i>.</span></li>
<li><span id="Mehta2023">Mehta,S. <i>et al.</i> (2023) A Galaxy of informatics resources for MS-based proteomics. <i>Expert Review of Proteomics</i>, 1–16.</span></li>
<li><span id="Mc Cartney2023.09.25.559365">The European Reference Genome Atlas: piloting a decentralised approach to equitable biodiversity genomics (2023) <i>bioRxiv</i>.</span></li>
<li><span id="Hrdtner2023">Härdtner,C. <i>et al.</i> (2023) A comparative gene expression matrix in Apoe-deficient mice identifies unique and atherosclerotic disease stage-specific gene regulation patterns in monocytes and macrophages. <i>Atherosclerosis</i>, <b>371</b>, 1–13.</span></li>
<li><span id="Rahman2023">Rahman,N. <i>et al.</i> (2023) Mobilisation and analyses of publicly available SARS-CoV-2 data for pandemic responses.</span></li>
<li><span id="Guerler2023">Guerler,A. <i>et al.</i> (2023) Fast and accurate genome-wide predictions and structural modeling of protein–protein interactions using Galaxy. <i>BMC Bioinformatics</i>, <b>24</b>.</span></li>
<li><span id="Riesle2023">Riesle,A.J. <i>et al.</i> (2023) Activator-blocker model of transcriptional regulation by pioneer-like factors. <i>Nature Communications</i>, <b>14</b>.</span></li>
<li><span id="Schiml2023">Schiml,V.C. <i>et al.</i> (2023) Integrative meta-omics in Galaxy and beyond. <i>Environmental Microbiome</i>, <b>18</b>.</span></li>
<li><span id="Hiltemann2023">Hiltemann,S. <i>et al.</i> (2023) Galaxy Training: A powerful framework for teaching! <i>PLOS Computational Biology</i>, <b>19</b>, e1010752.</span></li>
<li><span id="Bray2023">Bray,S. <i>et al.</i> (2023) The Planemo toolkit for developing,  deploying,  and executing scientific data analyses in Galaxy and beyond. <i>Genome Research</i>.</span></li></ol>
<h2 class="bibliography">2022</h2>
<ol class="bibliography"><li><span id="SoilandReyes2022">Soiland-Reyes,S. <i>et al.</i> (2022) Packaging research artefacts with RO-Crate. <i>Data Science</i>, <b>5</b>, 97–138.</span></li>
<li><span id="Rasche2022">Rasche,H. <i>et al.</i> (2022) Training Infrastructure as a Service. <i>GigaScience</i>, <b>12</b>.</span></li>
<li><span id="Kumar2022">Kumar,A. <i>et al.</i> (2022) An accessible infrastructure for artificial intelligence using a Docker-based JupyterLab in Galaxy. <i>GigaScience</i>, <b>12</b>.</span></li>
<li><span id="2022">Enis Afgan,and <i>et al.</i> (2022) The Galaxy platform for accessible,  reproducible and collaborative biomedical analyses: 2022 update. <i>Nucleic Acids Research</i>, <b>50</b>, W345–W351.</span></li>
<li><span id="Mehta2022">Mehta,S. <i>et al.</i> (2022) Catching the Wave: Detecting Strain-Specific SARS-CoV-2 Peptides in Clinical Samples Collected during Infection Waves from Diverse Geographical Locations. <i>Viruses</i>, <b>14</b>, 2205.</span></li>
<li><span id="Meier_2022">Meier,R. <i>et al.</i> (2022) The antileukemic activity of decitabine upon PML/RARA-negative AML blasts is supported by all-trans retinoic acid: in vitro and in vivo evidence for cooperation. <i>Blood Cancer Journal</i>, <b>12</b>.</span></li>
<li><span id="Wolff2022">Wolff,J. <i>et al.</i> (2022) Loop detection using Hi-C data with HiCExplorer. <i>GigaScience</i>, <b>11</b>.</span></li>
<li><span id="VijayKrishna2022">VijayKrishna,N. <i>et al.</i> (2022) Expanding the Galaxy’s reference data. <i>Bioinformatics Advances</i>, <b>2</b>.</span></li>
<li><span id="https://doi.org/10.11588/heibooks.979.c13724">Mühlhaus,T. <i>et al.</i> (2022) DataPLANT – Tools and Services to structure the Data Jungle for fundamental plant researchers.</span></li>
<li><span id="Pinter2022">Pinter,N. <i>et al.</i> (2022) MaxQuant and MSstats in Galaxy Enable Reproducible Cloud-Based Analysis of Quantitative Proteomics Experiments for Everyone. <i>Journal of Proteome Research</i>.</span></li>
<li><span id="Bray2022">Bray,S. <i>et al.</i> (2022) Galaxy workflows for fragment-based virtual screening: a case study on the SARS-CoV-2 main protease. <i>Journal of Cheminformatics</i>, <b>14</b>.</span></li>
<li><span id="Martin2022">Martin,D.P. <i>et al.</i> (2022) Selection analysis identifies clusters of unusual mutational changes in Omicron lineage BA.1 that likely impact Spike function. <i>Molecular Biology and Evolution</i>.</span></li>
<li><span id="Brack2022">Brack,P. <i>et al.</i> (2022) Ten simple rules for making a software tool workflow-ready. <i>PLOS Computational Biology</i>, <b>18</b>, e1009823.</span></li>
<li><span id="Mossad2022">Mossad,O. <i>et al.</i> (2022) Gut microbiota drives age-related oxidative stress and mitochondrial damage in microglia via the metabolite N6-carboxymethyllysine. <i>Nature Neuroscience</i>.</span></li>
<li><span id="Gao2022">Gao,M. <i>et al.</i> (2022) Pluripotency factors determine gene expression repertoire at zygotic genome activation. <i>Nature Communications</i>, <b>13</b>.</span></li>
<li><span id="Fahrner2022">Fahrner,M. <i>et al.</i> (2022) Democratizing data-independent acquisition proteomics analysis on public cloud infrastructures via the Galaxy framework. <i>GigaScience</i>, <b>11</b>.</span></li></ol>
<h2 class="bibliography">2021</h2>
<ol class="bibliography"><li><span id="Dai2021">Dai,C. <i>et al.</i> (2021) A proteomics sample metadata representation for multiomics integration and big data analysis. <i>Nature Communications</i>, <b>12</b>.</span></li>
<li><span id="mehta2021asaim">Mehta,S. <i>et al.</i> (2021) ASaiM-MT: a validated and optimized ASaiM workflow for metatranscriptomics analysis within Galaxy framework. <i>F1000Research</i>, <b>10</b>.</span></li>
<li><span id="batut2021rna">Batut,B. <i>et al.</i> (2021) RNA-Seq Data Analysis in Galaxy. In, <i>RNA Bioinformatics</i>. Springer, pp. 367–392.</span></li>
<li><span id="maier_ready--use_2021">Maier,W. <i>et al.</i> (2021) Ready-to-use public infrastructure for global SARS-CoV-2 monitoring. <i>Nature Biotechnology</i>, 1–2.</span></li>
<li><span id="Roncoroni2021">Roncoroni,M. <i>et al.</i> (2021) A SARS-CoV-2 sequence submission tool for the European Nucleotide Archive. <i>Bioinformatics</i>.</span></li>
<li><span id="Gu2021">Gu,Q. <i>et al.</i> (2021) Galaxy-ML: An accessible,  reproducible,  and scalable machine learning toolkit for biomedicine. <i>PLOS Computational Biology</i>, <b>17</b>, e1009014.</span></li>
<li><span id="rajczewski_rigorous_2021">Rajczewski,A.T. <i>et al.</i> (2021) A rigorous evaluation of optimal peptide targets for MS-based clinical diagnostics of Coronavirus Disease 2019 (COVID-19). <i>Clinical Proteomics</i>, <b>18</b>, 15.</span></li>
<li><span id="Wolff2021">Wolff,J. <i>et al.</i> (2021) Robust and efficient single-cell Hi-C clustering with approximate k-nearest neighbor graphs. <i>Bioinformatics</i>.</span></li>
<li><span id="10.1371/journal.pcbi.1008922">Gallardo-Alba,C. <i>et al.</i> (2021) A constructivist-based proposal for bioinformatics teaching practices during lockdown. <i>PLOS Computational Biology</i>, <b>17</b>, 1–11.</span></li>
<li><span id="10.1371/journal.pcbi.1008923">Serrano-Solano,B. <i>et al.</i> (2021) Fostering accessible online education using Galaxy as an e-learning platform. <i>PLOS Computational Biology</i>, <b>17</b>, 1–10.</span></li>
<li><span id="Moreno2021">Moreno,P. <i>et al.</i> (2021) User-friendly, scalable tools and workflows for single-cell RNA-seq analysis. <i>Nature Methods</i>.</span></li>
<li><span id="Bai2021">Bai,J. <i>et al.</i> (2021) BioContainers Registry: Searching Bioinformatics and Proteomics Tools,  Packages,  and Containers. <i>Journal of Proteome Research</i>.</span></li>
<li><span id="Videm2021">Videm,P. <i>et al.</i> (2021) ChiRA: an integrated framework for chimeric read analysis from RNA-RNA interactome and RNA structurome data. <i>GigaScience</i>, <b>10</b>.</span></li>
<li><span id="10.1093/gigascience/giaa152">Kumar,A. <i>et al.</i> (2021) Tool recommender system in Galaxy using deep learning. <i>GigaScience</i>, <b>10</b>.</span></li>
<li><span id="Wolff2023">Wolff,J. <i>et al.</i> (2021) Scool: a new data storage format for single-cell Hi-C data. <i>Bioinformatics</i>.</span></li></ol>
<h2 class="bibliography">2020</h2>
<ol class="bibliography"><li><span id="bray2020intuitive">Bray,S.A. <i>et al.</i> (2020) Intuitive, reproducible high-throughput molecular dynamics in Galaxy: a tutorial. <i>Journal of Cheminformatics</i>, <b>12</b>, 1–13.</span></li>
<li><span id="Greve2020">Greve,G. <i>et al.</i> (2020) Decitabine induces gene derepression on monosomic chromosomes: in vitro and in vivo effects in adverse-risk cytogenetics AML. <i>Cancer Research</i>, canres.1430.2020.</span></li>
<li><span id="marais2020genome">Marais,G.A.B. <i>et al.</i> (2020) Genome Evolution: Mutation Is the Main Driver of Genome Size in Prokaryotes. <i>Current Biology</i>, <b>30</b>, R1083–R1085.</span></li>
<li><span id="garcia2020ten">Garcia,L. <i>et al.</i> (2020) Ten simple rules for making training materials FAIR. <i>PLOS Computational Biology</i>, <b>16</b>, 1–9.</span></li>
<li><span id="Tekman2020">Tekman,M. <i>et al.</i> (2020) A single-cell RNA-sequencing training and analysis suite using the Galaxy framework. <i>GigaScience</i>, <b>9</b>.</span></li>
<li><span id="deKoning2020">de Koning,W. <i>et al.</i> (2020) NanoGalaxy: Nanopore long-read sequencing data analysis in Galaxy. <i>GigaScience</i>, <b>9</b>.</span></li>
<li><span id="Rasche2020">Rasche,H. and Gruening,B.A. (2020) Training Infrastructure as a Service.</span></li>
<li><span id="Baker2020">Baker,D. <i>et al.</i> (2020) No more business as usual: Agile and effective responses to emerging pathogen threats require open data and open analytics. <i>PLOS Pathogens</i>, <b>16</b>, e1008643.</span></li>
<li><span id="LopezDelisle2020">Lopez-Delisle,L. <i>et al.</i> (2020) pyGenomeTracks: reproducible plots for multivariate genomic data sets. <i>Bioinformatics</i>.</span></li>
<li><span id="Dass2020">Dass,G. <i>et al.</i> (2020) The omics discovery REST interface. <i>Nucleic Acids Research</i>, <b>48</b>, W380–W384.</span></li>
<li><span id="Wolff2020">Wolff,J. <i>et al.</i> (2020) Galaxy HiCExplorer 3: a web server for reproducible Hi-C,  capture Hi-C and single-cell Hi-C data analysis,  quality control and visualization. <i>Nucleic Acids Research</i>, <b>48</b>, W177–W184.</span></li>
<li><span id="Schfer2020">Schäfer,R.A. <i>et al.</i> (2020) GLASSGo in Galaxy: high-throughput,  reproducible and easy-to-integrate prediction of sRNA homologs. <i>Bioinformatics</i>.</span></li>
<li><span id="Bray2020">Bray,S.A. <i>et al.</i> (2020) The ChemicalToolbox: reproducible,  user-friendly cheminformatics analysis on the Galaxy platform. <i>Journal of Cheminformatics</i>, <b>12</b>.</span></li>
<li><span id="Hille2020">Hille,L. <i>et al.</i> (2020) Ultrastructural,  transcriptional and functional differences between human reticulated and non-reticulated platelets. <i>Journal of Thrombosis and Haemostasis</i>.</span></li>
<li><span id="Murat2020">Murat,K. <i>et al.</i> (2020) Ewastools: Infinium Human Methylation BeadChip pipeline for population epigenetics integrated into Galaxy. <i>GigaScience</i>, <b>9</b>.</span></li>
<li><span id="PerezRiverol2020">Perez-Riverol,Y. <i>et al.</i> (2020) CHAPTER 19. Cross-platform Software Development and Distribution with Bioconda and BioContainers. In, <i>Processing Metabolomics and Proteomics Data with Open Software</i>. Royal Society of Chemistry, pp. 415–426.</span></li>
<li><span id="Wolff2024">Wolff,J. <i>et al.</i> (2020) Loop detection using Hi-C data with HiCExplorer.</span></li>
<li><span id="Wolff2025">Wolff,J. <i>et al.</i> (2020) Approximate k-nearest neighbors graph for single-cell Hi-C dimensional reduction with MinHash.</span></li>
<li><span id="https://doi.org/10.11588/heibooks.598.c8418">Suchodoletz,D. von <i>et al.</i> (2020) Lessons learned from Virtualized Research Environments in today’s scientific compute infrastructures. <i>E-Science-Tage 2019</i>.</span></li>
<li><span id="Dass2021">Dass,G. <i>et al.</i> (2020) The omics discovery REST interface. <i>Nucleic Acids Research</i>.</span></li></ol>
<h2 class="bibliography">2019</h2>
<ol class="bibliography"><li><span id="Senapathi2019">Senapathi,T. <i>et al.</i> (2019) Biomolecular Reaction and Interaction Dynamics Global Environment (BRIDGE). <i>Bioinformatics</i>, <b>35</b>, 3508–3509.</span></li>
<li><span id="Gruening2019">Gruening,B. <i>et al.</i> (2019) Recommendations for the packaging and containerizing of bioinformatics software. <i>F1000Research</i>, <b>7</b>, 742.</span></li>
<li><span id="wibberg2019nbi">Wibberg,D. <i>et al.</i> (2019) The de. NBI/ELIXIR-DE training platform-Bioinformatics training in Germany and across Europe within ELIXIR. <i>F1000Research</i>, <b>8</b>.</span></li>
<li><span id="Scholz2019">Scholz,A. <i>et al.</i> (2019) uORF-Tools—Workflow for the determination of translation-regulatory upstream open reading frames. <i>PLOS ONE</i>, <b>14</b>, e0222459.</span></li>
<li><span id="Veil_2019">Veil,M. <i>et al.</i> (2019) Pou5f3, SoxB1, and Nanog remodel chromatin on High Nucleosome Affinity Regions at Zygotic Genome Activation. <i>Genome Research</i>, gr.240572.118.</span></li>
<li><span id="Walz_2019">Walz,J.M. <i>et al.</i> (2019) Impact of angiogenic activation and inhibition on miRNA profiles of human retinal endothelial cells. <i>Experimental Eye Research</i>.</span></li>
<li><span id="Fries_2019">Fries,A. <i>et al.</i> (2019) Alteration of the Route to Menaquinone towards Isochorismate-Derived Metabolites. <i>ChemBioChem</i>.</span></li>
<li><span id="10.1093/nar/gkz353">Fallmann,J. <i>et al.</i> (2019) The RNA workbench 2.0: next generation RNA data analysis. <i>Nucleic Acids Research</i>.</span></li>
<li><span id="Gruening2019Giga">Grüning,B.A. <i>et al.</i> (2019) Software engineering for scientific big data analysis. <i>GigaScience</i>, <b>8</b>.</span></li>
<li><span id="Ison2019">Ison,J. <i>et al.</i> (2019) The bio.tools registry of software tools and data resources for the life sciences. <i>Genome Biology</i>, <b>20</b>.</span></li>
<li><span id="Miladi2019">Miladi,M. <i>et al.</i> (2019) GraphClust2: Annotation and discovery of structured RNAs with scalable and accessible integrative clustering. <i>GigaScience</i>, <b>8</b>.</span></li>
<li><span id="Foell2019">Föll,M.C. <i>et al.</i> (2019) Accessible and reproducible mass spectrometry imaging data analysis in Galaxy. <i>GigaScience</i>, <b>8</b>.</span></li>
<li><span id="Wibberg2019">Wibberg,D. <i>et al.</i> (2019) The de.NBI / ELIXIR-DE training platform - Bioinformatics training in Germany and across Europe within ELIXIR. <i>F1000Research</i>, <b>8</b>, 1877.</span></li></ol>
<h2 class="bibliography">2018</h2>
<ol class="bibliography"><li><span id="batut2018asaim">Batut,B. <i>et al.</i> (2018) ASaiM: a Galaxy-based framework to analyze microbiota data. <i>GigaScience</i>, <b>7</b>, giy057.</span></li>
<li><span id="Anatskiy2018">Anatskiy,E. <i>et al.</i> (2018) Parkour LIMS: high-quality sample preparation in next generation sequencing. <i>Bioinformatics</i>, <b>35</b>, 1422–1424.</span></li>
<li><span id="Nhrenberg2018">Nührenberg,T.G. <i>et al.</i> (2018) Uncontrolled Diabetes Mellitus Has No Major Influence on the Platelet Transcriptome. <i>BioMed Research International</i>, <b>2018</b>, 1–9.</span></li>
<li><span id="Afgan2018">Afgan,E. <i>et al.</i> (2018) The Galaxy platform for accessible,  reproducible and collaborative biomedical analyses: 2018 update. <i>Nucleic Acids Research</i>, <b>46</b>, W537–W544.</span></li>
<li><span id="ramirez2018high">Ramı́rez Fidel <i>et al.</i> (2018) High-resolution TADs reveal DNA sequences underlying genome organization in flies. <i>Nature communications</i>, <b>9</b>, 189.</span></li>
<li><span id="gilsbach2018distinct">Gilsbach,R. <i>et al.</i> (2018) Distinct epigenetic programs regulate cardiac myocyte development and disease in the human heart in vivo. <i>Nature communications</i>, <b>9</b>, 391.</span></li>
<li><span id="thriene2018combinatorial">Thriene,K. <i>et al.</i> (2018) Combinatorial omics analysis reveals perturbed lysosomal homeostasis in collagen VII-deficient keratinocytes. <i>Molecular &amp; Cellular Proteomics</i>, mcp–RA117.</span></li>
<li><span id="blank2018disseminating">Blank,C. <i>et al.</i> (2018) Disseminating Metaproteomic Informatics Capabilities and Knowledge Using the Galaxy-P Framework. <i>Proteomes</i>, <b>6</b>, 7.</span></li>
<li><span id="Wolff2018">Wolff,J. <i>et al.</i> (2018) Galaxy HiCExplorer: a web server for reproducible Hi-C data analysis,  quality control and visualization. <i>Nucleic Acids Research</i>, <b>46</b>, W11–W16.</span></li>
<li><span id="Grning2018">Grüning,B. <i>et al.</i> (2018) Bioconda: sustainable and comprehensive software distribution for the life sciences. <i>Nature Methods</i>, <b>15</b>, 475–476.</span></li>
<li><span id="Grning2019">Grüning,B. <i>et al.</i> (2018) Practical Computational Reproducibility in the Life Sciences. <i>Cell Systems</i>, <b>6</b>, 631–635.</span></li>
<li><span id="Batut2018">Batut,B. <i>et al.</i> (2018) Community-Driven Data Analysis Training for Biology. <i>Cell Systems</i>, <b>6</b>, 752–758.e1.</span></li>
<li><span id="Argentini_2018">Argentini,A. <i>et al.</i> (2018) Update on the moFF Algorithm for Label-Free Quantitative Proteomics. <i>Journal of Proteome Research</i>.</span></li></ol>
<h2 class="bibliography">2017</h2>
<ol class="bibliography"><li><span id="Gruning_Rasche_Rebolledo-Jaramillo-Jupyt_and_Galax-2017">Gruning,B.A. <i>et al.</i> (2017) Jupyter and Galaxy: Easing entry barriers into complex
                  data analyses for biomedical researchers. <i>PLoS Comput Biol</i>, <b>13</b>, e1005425.</span></li>
<li><span id="Backofen_Engelhardt_Erxleben-RNA_Tools_servi-2017">Backofen,R. <i>et al.</i> (2017) RNA-bioinformatics: Tools, services and databases for
                  the analysis of RNA-based regulation. <i>J Biotechnol</i>.</span></li>
<li><span id="Gruning_Fallmann_Yusuf-The_RNA_workb-NAR2017">Gruning,B.A. <i>et al.</i> (2017) The RNA workbench: best practices for RNA and
                  high-throughput sequencing bioinformatics in Galaxy. <i>nar</i>.</span></li>
<li><span id="hornig2017grin3b">Hornig,T. <i>et al.</i> (2017) GRIN3B missense mutation as an inherited risk factor for schizophrenia: whole-exome sequencing in a family with a familiar history of psychotic disorders. <i>Genetics research</i>, <b>99</b>.</span></li>
<li><span id="wreczycka2017strategies">Wreczycka,K. <i>et al.</i> (2017) Strategies for analyzing bisulfite sequencing data. <i>Journal of biotechnology</i>, <b>261</b>, 105–115.</span></li>
<li><span id="da2017biocontainers">Veiga Leprevost,F. da <i>et al.</i> (2017) BioContainers: an open-source and community-driven framework for software standardization. <i>Bioinformatics</i>, <b>33</b>, 2580–2582.</span></li>
<li><span id="backofen2017rna">Backofen,R. <i>et al.</i> (2017) RNA-bioinformatics: tools, services and databases for the analysis of RNA-based regulation. <i>Journal of biotechnology</i>, <b>261</b>, 76–84.</span></li>
<li><span id="gruning2017rna">Grüning,B.A. <i>et al.</i> (2017) The RNA workbench: best practices for RNA and high-throughput sequencing bioinformatics in Galaxy. <i>Nucleic acids research</i>, <b>45</b>, W560–W566.</span></li>
<li><span id="jimenez2017four">Jiménez,R.C. <i>et al.</i> (2017) Four simple recommendations to encourage best practices in research software. <i>F1000Research</i>, <b>6</b>.</span></li>
<li><span id="walz2017mirna">Walz,J.M. <i>et al.</i> (2017) miRNA profile of human retinal endothelial cells in starvation or angiogenic stimulation with and without VEGF inhibitors. <i>Investigative Ophthalmology &amp; Visual Science</i>, <b>58</b>, 3567–3567.</span></li>
<li><span id="fallmann2017recent">Fallmann,J. <i>et al.</i> (2017) Recent advances in RNA folding. <i>Journal of biotechnology</i>, <b>261</b>, 97–104.</span></li>
<li><span id="batut2017building">Batut,B. <i>et al.</i> (2017) Building an open, collaborative, online infrastructure for bioinformatics training. <i>F1000Research</i>, <b>6</b>.</span></li>
<li><span id="doppelt2017regate">Doppelt-Azeroual,O. <i>et al.</i> (2017) ReGaTE, Registration of Galaxy Tools in Elixir. <i>Gigascience</i>.</span></li>
<li><span id="glaser2017ebf1">Glaser,L.V. <i>et al.</i> (2017) EBF1 binds to EBNA2 and promotes the assembly of EBNA2 chromatin complexes in B cells. <i>PLoS Pathogens</i>, <b>13</b>, e1006664.</span></li>
<li><span id="meier2017virtualisierte">Meier,K. <i>et al.</i> (2017) Virtualisierte wissenschaftliche Forschungsumgebungen und die zukünftige Rolle der Rechenzentren. <i>10. DFN-Forum Kommunikationstechnologien</i>.</span></li>
<li><span id="chambers2017accessible">Chambers,M.C. <i>et al.</i> (2017) An Accessible Proteogenomics Informatics Resource for Cancer Researchers. <i>Cancer research</i>, <b>77</b>, e43–e46.</span></li>
<li><span id="batut2017enasearch">Batut,B. and Grüning,B. (2017) ENASearch: A Python library for interacting with ENA’s API. <i>The Journal of Open Source Software</i>, <b>2</b>, 418.</span></li>
<li><span id="nothjunge2017dna">Nothjunge,S. <i>et al.</i> (2017) DNA methylation signatures follow preformed chromatin compartments in cardiac myocytes. <i>Nature communications</i>, <b>8</b>, 1667.</span></li></ol>
<h2 class="bibliography">2016</h2>
<ol class="bibliography"><li><span id="roidl2016dot1l">Roidl,D. <i>et al.</i> (2016) DOT1L Activity Promotes Proliferation and Protects Cortical Neural Stem Cells from Activation of ATF4-DDIT3-Mediated ER Stress In Vitro. <i>Stem Cells</i>, <b>34</b>, 233–245.</span></li>
<li><span id="wecker2016microrna">Wecker,T. <i>et al.</i> (2016) MicroRNA profiling in aqueous humor of individual human eyes by next-generation sequencing. <i>Investigative ophthalmology &amp; visual science</i>, <b>57</b>, 1706–1713.</span></li>
<li><span id="ramirez2016deeptools2">Ramı́rez Fidel <i>et al.</i> (2016) deepTools2: a next generation web server for deep-sequencing data analysis. <i>Nucleic acids research</i>, <b>44</b>, W160–W165.</span></li>
<li><span id="afgan2016galaxy">Afgan,E. <i>et al.</i> (2016) The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update. <i>Nucleic acids research</i>, <b>44</b>, W3–W10.</span></li>
<li><span id="huttel2016echinocandin">Hüttel,W. <i>et al.</i> (2016) Echinocandin B biosynthesis: a biosynthetic cluster from Aspergillus nidulans NRRL 8112 and reassembly of the subclusters Ecd and Hty from Aspergillus pachycristatus NRRL 11440 reveals a single coherent gene cluster. <i>BMC genomics</i>, <b>17</b>, 570.</span></li>
<li><span id="doring2016pubmedportable">Döring,K. <i>et al.</i> (2016) PubMedPortable: a framework for supporting the development of text mining applications. <i>PloS one</i>, <b>11</b>, e0163794.</span></li>
<li><span id="kranzhofer20165">Kranzhöfer,D.K. <i>et al.</i> (2016) 5’-hydroxymethylcytosine precedes loss of CpG methylation in enhancers and genes undergoing activation in cardiomyocyte maturation. <i>PloS one</i>, <b>11</b>, e0166575.</span></li></ol>
<h2 class="bibliography">2015</h2>
<ol class="bibliography"><li><span id="cock2015ncbi">Cock,P.J.A. <i>et al.</i> (2015) NCBI BLAST+ integrated into Galaxy. <i>Gigascience</i>, <b>4</b>, 39.</span></li>
<li><span id="lucas2015purchasable">Lucas,X. <i>et al.</i> (2015) The purchasable chemical space: a detailed picture. <i>Journal of chemical information and modeling</i>, <b>55</b>, 915–924.</span></li>
<li><span id="preissl2015deciphering">Preissl,S. <i>et al.</i> (2015) Deciphering the Epigenetic Code of Cardiac Myocyte TranscriptionNovelty and Significance. <i>Circulation research</i>, <b>117</b>, 413–423.</span></li>
<li><span id="yachdav2015cutting">Yachdav,G. <i>et al.</i> (2015) Cutting edge: anatomy of BioJS, an open source community for the life sciences. <i>Elife</i>, <b>4</b>, e07009.</span></li>
<li><span id="gilsbach2015genome">Gilsbach,R. <i>et al.</i> (2015) Genome wide epigenetic profiling of purified cardiomyocytes enables deep insights into gene expression control. <i>NAUNYN-SCHMIEDEBERGS ARCHIVES OF PHARMACOLOGY</i>, <b>388</b>, S55–S55.</span></li>
<li><span id="preissl2015dynamics">Preissl,S. <i>et al.</i> (2015) Dynamics of epigenetic modifications in cardiomyocyte-specific cis-regulatory regions. <i>NAUNYN-SCHMIEDEBERGS ARCHIVES OF PHARMACOLOGY</i>, <b>388</b>, S55–S56.</span></li>
<li><span id="ison2015tools">Ison,J. <i>et al.</i> (2015) Tools and data services registry: a community effort to document bioinformatics resources. <i>Nucleic acids research</i>, <b>44</b>, D38–D47.</span></li>
<li><span id="li2015ngs">Li,J. <i>et al.</i> (2015) An NGS workflow blueprint for DNA sequencing data and its application in individualized molecular oncology. <i>Cancer informatics</i>, <b>14</b>, CIN–S30793.</span></li>
<li><span id="gruning2015integrierte">Grüning,B. (2015) Integrierte bioinformatische Methoden zur reproduzierbaren und transparenten Hochdurchsatz-Analyse von Life Science Big Data.</span></li></ol>
<h2 class="bibliography">2014</h2>
<ol class="bibliography"><li><span id="lucas2014chemicaltoolbox">Lucas,X. <i>et al.</i> (2014) ChemicalToolBoX and its application on the study of the drug like and purchasable space. <i>Journal of Cheminformatics</i>, <b>6</b>, P51.</span></li>
<li><span id="telukunta2014dynamic">Telukunta,K.K. <i>et al.</i> (2014) Dynamic information system for small molecules. <i>Journal of cheminformatics</i>, <b>6</b>, P28.</span></li>
<li><span id="ramirez2014deeptools">Ramı́rez Fidel <i>et al.</i> (2014) deepTools: a flexible platform for exploring deep-sequencing data. <i>Nucleic acids research</i>, <b>42</b>, W187–W191.</span></li>
<li><span id="patel2014pywater">Patel,H. <i>et al.</i> (2014) PyWATER: a PyMOL plug-in to find conserved water molecules in proteins by clustering. <i>Bioinformatics</i>, <b>30</b>, 2978–2980.</span></li>
<li><span id="gilsbach2014dynamic">Gilsbach,R. <i>et al.</i> (2014) Dynamic DNA methylation orchestrates cardiomyocyte development, maturation and disease. <i>Nature communications</i>, <b>5</b>, 5288.</span></li>
<li><span id="schubert2014autosomal">Schubert,D. <i>et al.</i> (2014) Autosomal dominant immune dysregulation syndrome in humans with CTLA4 mutations. <i>Nature medicine</i>, <b>20</b>, 1410.</span></li>
<li><span id="volk2014autosomal">Volk,T. <i>et al.</i> (2014) Autosomal-Recessive Agammaglobulinemia Due to Homozygous Mutations in Artemis: Do We Need a Modifier? <i>JOURNAL OF CLINICAL IMMUNOLOGY</i>, <b>34</b>, S146–S147.</span></li>
<li><span id="schubert2014ctla">Schubert,D. <i>et al.</i> (2014) CTLA-4 Deficiency-A Novel Autosomal-Dominant Immune Dysregulation Syndrome. <i>JOURNAL OF CLINICAL IMMUNOLOGY</i>, <b>34</b>, S144–S144.</span></li>
<li><span id="bulashevska2014bioinformatics">Bulashevska,A. <i>et al.</i> (2014) Bioinformatics Analysis of Exome Sequencing Data: Challenges and Solutions. <i>JOURNAL OF CLINICAL IMMUNOLOGY</i>, <b>34</b>, S328–S328.</span></li>
<li><span id="preissl2014cpg">Preissl,S. <i>et al.</i> (2014) CpG-methylation characterizes cardiomyocytes in development and disease. <i>NAUNYN-SCHMIEDEBERGS ARCHIVES OF PHARMACOLOGY</i>, <b>387</b>, S76–S76.</span></li></ol>
<h2 class="bibliography">2013</h2>
<ol class="bibliography"><li><span id="gruning2013draft">Grüning,B.A. <i>et al.</i> (2013) Draft genome sequence of Streptomyces viridochromogenes strain Tü57, producer of avilamycin. <i>Genome announcements</i>, <b>1</b>, e00384–13.</span></li>
<li><span id="cock2013galaxy">Cock,P.J.A. <i>et al.</i> (2013) Galaxy tools and workflows for sequence analysis with applications in molecular plant pathology. <i>PeerJ</i>, <b>1</b>, e167.</span></li>
<li><span id="youssar2013characterization">Youssar,L. <i>et al.</i> (2013) Characterization and phylogenetic analysis of the mitochondrial genome of Glarea lozoyensis indicates high diversity within the order Helotiales. <i>PloS one</i>, <b>8</b>, e74792.</span></li></ol>
<h2 class="bibliography">2012</h2>
<ol class="bibliography"><li><span id="youssar2012genome">Youssar,L. <i>et al.</i> (2012) Genome sequence of the fungus Glarea lozoyensis: the first genome sequence of a species from the Helotiaceae family. <i>Eukaryotic cell</i>, <b>11</b>, 250–250.</span></li>
<li><span id="senger2012mining">Senger,C. <i>et al.</i> (2012) Mining and evaluation of molecular relationships in literature. <i>Bioinformatics</i>, <b>28</b>, 709–714.</span></li>
<li><span id="lucas2012streptomedb">Lucas,X. <i>et al.</i> (2012) StreptomeDB: a resource for natural compounds isolated from Streptomyces species. <i>Nucleic acids research</i>, <b>41</b>, D1130–D1136.</span></li>
<li><span id="gunther2012genome">Günther,S. <i>et al.</i> (2012) Genome Sequence of the Fungus Glarea. <i>Eukaryotic Cell</i>, <b>11</b>, 250.</span></li></ol>
<h2 class="bibliography">2011</h2>
<ol class="bibliography"><li><span id="erxleben2011genome">Erxleben,A. <i>et al.</i> (2011) Genome sequence of Streptomyces sp. Tü6071. <i>Journal of bacteriology</i>, JB–00377.</span></li>
<li><span id="gruning2011compounds">Grüning,B.A. <i>et al.</i> (2011) Compounds In Literature (CIL): screening for compounds and relatives in PubMed. <i>Bioinformatics</i>, <b>27</b>, 1341–1342.</span></li>
<li><span id="bieschke2011small">Bieschke,J. <i>et al.</i> (2011) Small-molecule conversion of toxic oligomers to nontoxic β-sheet–rich amyloid fibrils. <i>Nature Chemical Biology</i>, <b>8</b>, 93–101.</span></li></ol>
<h2 class="bibliography">2009</h2>
<ol class="bibliography"><li><span id="hildebrand2009superlooper">Hildebrand,P.W. <i>et al.</i> (2009) SuperLooper—a prediction server for the modeling of loops in globular and membrane proteins. <i>Nucleic acids research</i>, <b>37</b>, W571–W574.</span></li>
<li><span id="rose2009rhythm">Rose,A. <i>et al.</i> (2009) RHYTHM—a server to predict the orientation of transmembrane helices in channels and membrane-coils. <i>Nucleic acids research</i>, <b>37</b>, W575–W580.</span></li></ol>
<h2 class="bibliography">2008</h2>
<ol class="bibliography"><li><span id="rother2008voronoia">Rother,K. <i>et al.</i> (2008) Voronoia: analyzing packing in protein structures. <i>Nucleic acids research</i>, <b>37</b>, D393–D395.</span></li>
<li><span id="dunkel2008superscent">Dunkel,M. <i>et al.</i> (2008) SuperScent—a database of flavors and scents. <i>Nucleic acids research</i>, <b>37</b>, D291–D294.</span></li>
<li><span id="schmidt2008supertoxic">Schmidt,U. <i>et al.</i> (2008) SuperToxic: a comprehensive database of toxic compounds. <i>Nucleic acids research</i>, <b>37</b>, D295–D299.</span></li>
<li><span id="struck2008toxicity">Struck,S. <i>et al.</i> (2008) Toxicity versus potency: Elucidation of toxicity properties discriminating between toxins, drugs, and natural compounds. <i>Genome Informatics 2008: Genome Informatics Series Vol. 20</i>, 231–242.</span></li>
<li><span id="struck2008properties">STRUCK,S.W.A.N.T.J.E. <i>et al.</i> (2008) PROPERTIES DISCRIMINATING BETWEEN TOXINS, DRUGS. <i>Genome Informatics 2008: Genome Informatics Series Vol. 20</i>, 231.</span></li></ol>

        </div>
      </div>
    </div>
    







<footer class="navbar-default">
    <div class="container">
        <div class="row">
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <p>The <a href="/freiburg/people">Freiburg Galaxy Team</a> is headed by Björn Grüning
and is part of the <a href="http://www.bioinf.uni-freiburg.de">Bioinformatics Group
Freiburg</a>, headed by Prof. Dr. Rolf
Backofen.</p>

<p>The team is part of <a href="https://www.denbi.de">de.NBI</a> and
<a href="https://www.elixir-europe.org">ELIXIR</a> and largely funded by
<a href="http://dfg.de/">DFG</a> and <a href="https://www.bmbf.de">BMBF</a>. All content on
this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a>,
unless otherwise specified.</p>

            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu/website/tree/master/freiburg/publications.md" target="_blank">Edit this page on GitHub</a></li>
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:contact@usegalaxy.eu">contact@usegalaxy.eu</a></li>
                    
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu" target="_blank">usegalaxy-eu</a></li>
                    
                    
                    
                      <li><img class="fa-mastodon" src="/assets/media/mastodon.svg" style="width:18px;height:18px;padding-right:4px;filter:grayscale(100%);-webkit-filter:grayscale(100%);"/><a href="https://bawü.social/@galaxyfreiburg" target="_blank">galaxyfreiburg</a></li>
                    
                      <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li>
                </ul>
            </div>
        </div>
    </div>
</footer>

<script async defer data-domain="galaxyproject.eu" src="https://plausible.galaxyproject.eu/js/plausible.js"></script>
  </body>
</html>
